Detected,Related to Tensor,Time,Version,CUDA,Category Impact,Impact,Buggy API,Log Rule,Log Rule 2,Log Message,Issue,Category,Trigger,Need More Inspection,Reason not agree
FuzzGPT,yes,2023-04-16T13:44:32Z,2.0.0,cu118,Logical,TypeError,torch.addcmul,invalid combination,N\A,TypeError: addcmul() received an invalid combination of arguments - got unrecognized keyword arguments: tensor2,https://github.com/pytorch/pytorch/issues/99265,API Input Context,Modeling Parameter Space,,
,no,2023-06-01T14:01:57Z,2.0.0,cu118,Logical,Inconsistent results,torch.distributed.fsdp.FullyShardedDataParallel,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/102731,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-26T12:44:11Z,2.0.0,cu117,Logical,Inconsistent results,torch.nn.TransformerEncoderLayer,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/100087,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-05-03T21:29:21Z,2.0.0,cu117,Logical,RuntimeError,torch.nn.parallel.DistributedDataParallel,Expected to,N\A,RuntimeError('Expected to mark a variable ready only once. This error is caused [...],https://github.com/pytorch/pytorch/issues/100582,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-04-24T22:17:50Z,2.0.0,cu117,Logical,Others,torch.distributed.fsdp.FullyShardedDataParallel,unexpected,N\A,An unexpected prefix is detected,https://github.com/pytorch/pytorch/issues/99932,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-08-04T21:04:36Z,2.0.0,N.A,Logical,AssertionError,torch.utils.data.DataLoader,assert,N\A,"    assert self._timeout == 0
AssertionError",https://github.com/pytorch/pytorch/issues/106634,API Input Context,Modeling Parameter Space,,
,no,2023-06-14T06:34:55Z,2.0.0,N.A,Performance bug,Performance bug,torch.utils.data.DataLoader,Performance bug,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/103581,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-04-10T22:32:05Z,2.0.0,N.A,Logical,RuntimeError,torch.utils.data.DataLoader,Expected a,N\A,# RuntimeError: Expected a 'cpu' device type for generator but found 'cuda',https://github.com/pytorch/pytorch/issues/98792,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2023-07-03T20:02:34Z,2.0.0,N.A,Logical,RuntimeError,torch.compile,Failed running,N\A,"RuntimeError: Failed running call_function <function get_div_fn.<locals>._f at 0x7f8278b018b0>(*(FakeTensor(FakeTensor(..., device='meta', size=(3,)), cpu), FakeTensor(FakeTensor(..., device='meta', size=(1,)), cpu)), **{}):",https://github.com/pytorch/pytorch/issues/104536,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-01-30T05:04:16Z,2.0.0,cu117,Logical,AssertionError,torch.Tensor.min,only supports,N\A,AssertionError: float16 user-defined reduction only supports 'sum' and 'any' but got min,https://github.com/pytorch/pytorch/issues/93249,API Input Context,Modeling Specific Data Types,,
FuzzGPT,yes,2023-02-21T06:56:13Z,2.0.0,CPU,Logical,RuntimeError,torch.Tensor.as_strided_,requiring a storage,N\A,"RuntimeError: setStorage: sizes [2, 2], strides [2, 2], storage offset 7, and itemsize 4 requiring a storage size of 48 are out of bounds for storage of size 20",https://github.com/pytorch/pytorch/issues/95186,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2023-04-12T16:53:22Z,2.0.0,cu118,Logical,TypeError,torch.tensor,len(),N\A,TypeError: len() of a 0-d tensor,https://github.com/pytorch/pytorch/issues/98948,API Input Context,Modeling Parameter Space,,
,yes,2023-04-12T09:54:46Z,2.0.0,cu117,Logical,RuntimeError,torch.sparse_csr_tensor,does not require,N\A,RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn,https://github.com/pytorch/pytorch/issues/98929,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-03-27T13:54:17Z,2.0.0,cu117,Logical,Inconsistent results,torch.sparse_csr_tensor,Sparse CSR tensor,N\A,"float32
/home/robert/test/min_bfloat16_fail.py:30: UserWarning: Sparse CSR tensor support is in beta state. If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues. (Triggered internally at ../aten/src/ATen/SparseCsrTensorImpl.cpp:54.)
  out = torch.sparse_csr_tensor(
  Max forward abs diff:  tensor(7.6294e-06, device='cuda:0')
  Max forward rel diff:  tensor(1.8543e-05, device='cuda:0')
float16
  Max forward abs diff:  tensor(0.0312, device='cuda:0', dtype=torch.float16)
  Max forward rel diff:  tensor(0.1309, device='cuda:0', dtype=torch.float16)
bfloat16
  Max forward abs diff:  tensor(0.2500, device='cuda:0', dtype=torch.bfloat16)
  Max forward rel diff:  tensor(2.0156, device='cuda:0', dtype=torch.bfloat16)",https://github.com/pytorch/pytorch/issues/97668,API Input Context,Modeling Specific Data Types,,
,yes,2023-04-27T09:27:21Z,2.0.0,cu117,Logical,RuntimeError,torch.sparse_csc_tensor,MKL error,N\A,"RuntimeError: MKL error: SPARSE_STATUS_ALLOC_FAILED when calling `mkl_sparse_spmm( SPARSE_OPERATION_NON_TRANSPOSE, mkl_sparse_mat1.descriptor(), mkl_sparse_mat2.descriptor(), &result_desc)`",https://github.com/pytorch/pytorch/issues/100158,API Input Context,Large Integer Argument,,
,yes,2023-03-20T21:57:52Z,2.0.0,cu117,Logical,RuntimeError,torch.randint,Overflow,N\A,RuntimeError: Overflow when unpacking long,https://github.com/pytorch/pytorch/issues/97189,API Input Context,Modeling Specific Data Types,,
,no,2023-04-12T07:57:32Z,2.0.0,cu117,Logical,Others,torch.optim.lr_scheduler.MultiStepLR,Weights only load,N\A,"_pickle.UnpicklingError: Weights only load failed. Re-running `torch.load` with `weights_only` set to `False` will likely succeed, but it can result in arbitrary code execution.Do it only if you get the file from a trusted source. WeightsUnpickler error: Unsupported class collections.Counter",https://github.com/pytorch/pytorch/issues/98921,API Input Context,Modeling Parameter Space,,
,yes,2023-01-20T08:15:59Z,2.0.0,cu117,Logical,RuntimeError,torch.nn.BatchNorm2d,must match,N\A,RuntimeError: The size of tensor a (3) must match the size of tensor b (2) at non-singleton dimension 0,https://github.com/pytorch/pytorch/issues/92674,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-03-31T22:41:15Z,2.0.0,cu117,Logical,RuntimeError,torch.onnx.export,Input must be,N\A,RuntimeError: Input must be a list of dictionaries or a single numpy array for input 'x'.,https://github.com/pytorch/pytorch/issues/98100,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-03-23T18:57:19Z,2.0.0,CPU,Logical,RuntimeError,torch.onnx.export,Only tuples,N\A,"RuntimeError: Only tuples, lists and Variables are supported as JIT inputs/outputs. Dictionaries and strings are also accepted, but their usage is not recommended. Here, received an input of unsupported type: torch._C.ScriptObject",https://github.com/pytorch/pytorch/issues/97456,API Input Context,Modeling Parameter Space,,
,no,2023-03-16T21:19:53Z,2.0.0,N.A,Logical,TypeError,torch.onnx.export,positional argument,N\A,TypeError: MyModel.forward() takes 1 positional argument but 2 were given,https://github.com/pytorch/pytorch/issues/96981,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-04-26T12:40:00Z,2.0.0,cu11,Logical,RuntimeError,torch.Tensor.norm,Expected out tensor,N\A,"RuntimeError: Expected out tensor to have dtype float, but got c10::BFloat16 instead",https://github.com/pytorch/pytorch/issues/100085,API Input Context,Modeling Specific Data Types,,
,no,2023-03-31T07:13:17Z,2.0.0,cu117,Logical,RuntimeError,torch.nn.Sequential,has no attribute,N\A,RuntimeError: Module 'Sequential' has no attribute '_modules' :,https://github.com/pytorch/pytorch/issues/98064,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-07-06T10:23:34Z,2.0.0,cu117,Logical,Inconsistent results,torch.nn.LSTM,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/104697,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-02-01T05:27:07Z,2.0.0,cu117,Logical,BackendCompilerFailed,torch.nn.functional.log_softmax,assert,N\A,"    assert all(index[i] == 0 for i in reduced_idx)
AssertionError",https://github.com/pytorch/pytorch/issues/93819,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-07-13T08:08:47Z,2.0.0,cu117,Logical,TypeError,torch.onnx.export,not subscriptable,N\A,TypeError: 'NoneType' object is not subscriptable (Occurred when translating col2im).,https://github.com/pytorch/pytorch/issues/105134,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-01-21T03:41:21Z,2.0.0,cu117,Logical,RuntimeError,torch.nn.Conv2d,should be the same,N\A,RuntimeError: Input type (torch.FloatTensor) and weight type (Mkldnntorch.FloatTensor) should be the same,https://github.com/pytorch/pytorch/issues/92740,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-01-20T08:15:59Z,2.0.0,cu117,Logical,RuntimeError,torch.nn.Linear,The size of tensor a,N\A,RuntimeError: The size of tensor a (3) must match the size of tensor b (2) at non-singleton dimension 0,https://github.com/pytorch/pytorch/issues/92674,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-03T05:58:40Z,2.0.0,N.A,Logical,Inconsistent results,torch.onnx.export,graph(,N\A,"Exported graph: graph(%input : Float(1, 16, 400, 640, strides=[4096000, 256000, 640, 1], requires_grad=0, device=cuda:0),",https://github.com/pytorch/pytorch/issues/98189,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
TitanFuzz,yes,2023-02-22T01:57:02Z,2.0.0,cu117,Logical,Name Error,torch.Tensor.moveaxis,is not defined,N\A,NameError: name 'buf0' is not defined,https://github.com/pytorch/pytorch/issues/95262,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-04-08T16:29:08Z,2.0.0,cu117,Logical,RuntimeError,torch.Tensor.matmul,Sparse CSR tensors,N\A,RuntimeError: Sparse CSR tensors do not have strides,https://github.com/pytorch/pytorch/issues/98675,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-03-16T17:01:27Z,2.0.0,N.A,Logical,Others,torch.Tensor.matmul,Cannot read,N\A,rocBLAS error: Cannot read /home/kali/miniconda3/envs/pytorch2/lib/python3.9/site-packages/torch/lib/rocblas/library/TensileLibrary.dat: No such file or directory,https://github.com/pytorch/pytorch/issues/96964,Test Case Context,Model Tensor Execution on CUDA Devices,,
TitanFuzz,yes,2023-02-02T19:44:24Z,2.0.0,cu117,Logical,RuntimeError,torch.linalg.lstsq,shapes cannot,N\A,# RuntimeError: mat1 and mat2 shapes cannot be multiplied (6x3 and 2x3),https://github.com/pytorch/pytorch/issues/93982,API Input Context,Modeling Parameter Space,,
,no,2023-04-03T13:27:17Z,2.0.0,N.A,Logical,Inconsistent results,torch.jit.script,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/98210,Test Case Context,Modeling Compile/Eager Execution Mode,,
,no,2022-12-29T13:04:39Z,2.0.0,N.A,Logical,RuntimeError,torch.jit.script,Can't redefine method,N\A,RuntimeError: Can't redefine method: forward on class: __torch__.SoSadModule (of Python compilation unit at: 0x3f45600),https://github.com/pytorch/pytorch/issues/91504,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-01-24T18:24:24Z,2.0.0,N.A,Logical,Inconsistent results,torch.jit.save,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/92910,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-03-29T21:40:31Z,2.0.0,N.A,Logical,ValueError,torch.jit.annotate,Cannot determine scalar type,N\A,"torch.onnx.errors.SymbolicValueError: Cannot determine scalar type for this '<class 'torch.TensorType'>' instance and a default value was not provided.  [Caused by the value '21 defined in (%21 : Tensor = onnx::Sub(%input, %20), scope: __main__.Example::/torch.nn.modules.normalization.LayerNorm::layer_norm",https://github.com/pytorch/pytorch/issues/97913,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,no,2023-10-13T21:53:59Z,2.0.0,cu117,Logical,Performance bug,torch.index_select,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/111247,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-02-21T07:35:18Z,2.0.0,N.A,Logical,Others,torch.index_add,compile error,N\A,torch._inductor.exc.CppCompileError: C++ compile error,https://github.com/pytorch/pytorch/issues/95189,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-05-23T13:50:21Z,2.0.0,cu117,Logical,Others,torch.fx.symbolic_trace,symbolically traced variables,N\A,TraceError: symbolically traced variables cannot be used as inputs to control flow,https://github.com/pytorch/pytorch/issues/102085,Test Case Context,Modeling Multiple API Usage Pattern,,
,no,2023-04-19T19:09:46Z,2.0.0,N.A,Logical,RuntimeError,torch.func.jacrev,In order to use an,N\A,"RuntimeError: In order to use an autograd.Function with functorch transforms (vmap, grad, jvp, jacrev, ...), it must override the setup_context staticmethod. For more details, please see https://pytorch.org/docs/master/notes/extending.func.html",https://github.com/pytorch/pytorch/issues/99556,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-06-07T17:18:19Z,2.0.0,cu118,Logical,RuntimeError,torch.func.functional_call.jacrev,Cannot access data,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/103161,Test Case Context,Modeling Multiple API Usage Pattern,,
,no,2023-03-29T20:54:51Z,2.0.0,cu117,Logical,Inconsistent results,torch.func.functional_call,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/97909,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-09-17T00:24:34Z,2.0.0,N.A,Logical,Inconsistent results,torch.fft.fft,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109448,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-05-12T02:59:52Z,2.0.0,cu121,Performance bug,Performance bug,torch.einsum,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/101249,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-05-08T18:09:01Z,2.0.0,cu117,Logical,Inconsistent results,torch.distributions.categorical.Categorical,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/100884,API Input Context,Modeling Parameter Space,,
,yes,2023-10-01T20:44:04Z,2.0.0,cu117,Logical,ValueError,torch.distributions.categorical.Categorical,to satisfy the constraint,N\A,"Expected parameter probs (Tensor of shape (256,)) of distribution Categorical(probs: torch.Size([256])) to satisfy the constraint Simplex(), but found invalid values:",https://github.com/pytorch/pytorch/issues/110366,API Input Context,Modeling Parameter Space,,
,no,2023-07-05T08:36:54Z,2.0.0,CPU,Logical,AssertionError,torch.distributed.rpc.backend_registry.register_backend,assert,N\A,"Traceback (most recent call last):
  File ""/localdata/georgew/scratch/my_rpc_backend.py"", line 27, in <module>
    assert hasattr(rpc.BackendType, ""MY_BACKEND_NAME"")  # fails
AssertionError",https://github.com/pytorch/pytorch/issues/104620,API Input Context,Modeling Parameter Space,,
,no,2023-07-02T07:33:57Z,2.0.0,cu117,Performance bug,Performance bug,torch.distributed.init_process_group,Hang,N\A,Hang,https://github.com/pytorch/pytorch/issues/104506,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-05-09T01:26:39Z,2.0.0,N.A,Logical,RuntimeError,torch.concat,Unexpected floating,N\A,RuntimeError: Unexpected floating ScalarType in at::autocast::prioritize,https://github.com/pytorch/pytorch/issues/100932,API Input Context,Modeling Specific Data Types,,
,yes,2023-01-01T11:14:12Z,2.0.0,CPU,Performance bug,Performance bug,torch.compile,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/93498,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-04-28T08:55:57Z,2.0.0,cu117,Logical,Others,torch.compile,raise,N\A,"raise ProcessRaisedException(msg, error_index, failed_process.pid)
torch.multiprocessing.spawn.ProcessRaisedException:",https://github.com/pytorch/pytorch/issues/100239,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-04-11T09:47:02Z,2.0.0,N.A,Vulnerability,Segmentation fault,torch.compile,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/98835,API Input Context,Empty Input Tensor,,
,no,2023-03-30T16:45:08Z,2.0.0,cu117,Logical,AssertionError,torch.compile,daemonic processes,N\A,AssertionError: daemonic processes are not allowed to have children,https://github.com/pytorch/pytorch/issues/97992,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,no,2023-03-28T15:15:01Z,2.0.0,cu117,Logical,RuntimeError,torch.compile,Failed running,N\A,"RuntimeError: Failed running call_module model(*(FakeTensor(FakeTensor(..., device='meta', size=(1, 100)), cpu),), **{}):",https://github.com/pytorch/pytorch/issues/97784,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-03-23T11:55:13Z,2.0.0,cu117,Logical,AssertionError,torch.compile,assert,N\A,"    assert ""source"" in options and options[""source""] is not None
AssertionError: ",https://github.com/pytorch/pytorch/issues/97436,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-03-22T07:13:46Z,2.0.0,cu117,Logical,BackendCompilerFailed,torch.compile,raised,N\A,"torch._dynamo.exc.BackendCompilerFailed: debug_wrapper raised LoweringException: AssertionError: Pointwise( 'cuda',",https://github.com/pytorch/pytorch/issues/97340,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-02-14T20:27:40Z,2.0.0,cu117,Logical,RuntimeError,torch.func.grad,Cannot access,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/94851,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-02-13T22:24:56Z,2.0.0,cu117,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/94773,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-01-31T19:17:07Z,2.0.0,cu117,Logical,BackendCompilerFailed,torch.compile,object has no attribute,N\A,torch._dynamo.exc.BackendCompilerFailed: debug_wrapper raised AttributeError: 'ExpandView' object has no attribute 'make_indexer',https://github.com/pytorch/pytorch/issues/93386,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-01-30T11:09:57Z,2.0.0,cu117,Logical,AssertionError,torch.compile,AssertionError,N\A,AssertionError,https://github.com/pytorch/pytorch/issues/93262,Test Case Context+API Input Context,Modeling Compile/Eager Execution Mode+Modeling Parameter Space,,
,yes,2022-12-31T01:40:33Z,2.0.0,cu118,Logical,RuntimeError,torch.compile,Trying to backward,N\A,RuntimeError: Trying to backward through the graph a second time (or directly access saved tensors after they have already been freed). Saved intermediate values of the graph are freed when you call .backward() or autograd.grad(). Specify retain_graph=True if you need to backward through the graph a second time or if you need to access saved tensors after calling backward.,https://github.com/pytorch/pytorch/issues/91542,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2022-12-13T17:28:48Z,2.0.0,N.A,Logical,Others,torch.compile,argument of type,N\A,NotImplementedError: argument of type: <class 'typing._GenericAlias'>,https://github.com/pytorch/pytorch/issues/90774,Test Case Context,Supporting External Library API,,
,yes,2023-01-08T23:31:55Z,2.0.0,N.A,Logical,Inconsistent results,torch.distributions.Categorical,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/91863,API Input Context,Modeling Parameter Space,,
,yes,2023-04-11T19:00:20Z,2.0.0,cu121,Logical,RuntimeError,torch.cat,Sparse CSR tensors,N\A,RuntimeError: Sparse CSR tensors do not have is_contiguous,https://github.com/pytorch/pytorch/issues/98861,API Input Context,Modeling Parameter Space,,
,yes,2023-03-07T10:40:44Z,2.0.0,N.A,Logical,ValueError,torch.autograd.gradgradcheck,got an unexpected keyword,N\A,TypeError: gradgradcheck() got an unexpected keyword argument 'check_sparse_nnz',https://github.com/pytorch/pytorch/issues/96187,API Input Context,Modeling Parameter Space,,
,yes,2023-08-03T16:41:36Z,2.0.0,CPU,Logical,Others,torch.autograd.grad,bad operand,N\A,bad operand type for unary -: 'NoneType',https://github.com/pytorch/pytorch/issues/106556,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
ACETest,yes,2023-02-10T13:28:31Z,2.0.0,cu118,Vulnerability,Segmentation fault,torch.affine_grid_generator,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94590,API Input Context,Large Input Tensor,,
"TitanFuzz, DeepRel",yes,2023-01-21T03:55:09Z,2.0.0,cu117,Logical,RuntimeError,torch.Tensor.addmm,The size of tensor a,N\A,RuntimeError:  The size of tensor a (64) must match the size of tensor b (32) at non-singleton dimension 0,https://github.com/pytorch/pytorch/issues/92742,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
ACETest,yes,2023-02-10T13:46:52Z,2.0.0,cu118,Vulerability,InvalidPointer,torch._remove_batch_dim,Aborted (core dumped),Aborted,"free(): invalid pointer
Aborted (core dumped)",https://github.com/pytorch/pytorch/issues/94593,API Input Context,Negative Integer Argument,,
,yes,2022-12-15T15:11:28Z,2.0.0,CPU,Logical,Inconsistent results,torch._dynamo.optimizations.training.aot_autograd,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/90923,Test Case Context,Modeling Compile/Eager Execution Mode,,
,no,2023-03-18T08:34:15Z,2.0.0,cu117,Logical,Graph break,torch._dynamo.explain, missing:,N\A,torch._dynamo.exc.Unsupported: missing: POP_FINALLY,https://github.com/pytorch/pytorch/issues/97084,Test Case Context,Supporting External Library API,,
,yes,2023-02-10T13:37:37Z,2.0.0,cu116,Logical,IndexError,torch._C._nn.upsample_nearest2d,out of bounds for,N\A,IndexError: index 2 is out of bounds for dimension 0 with size 2,https://github.com/pytorch/pytorch/issues/94591,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-03-20T06:01:36Z,2.0.0,cu118,Logical,TypeError,torch.nn.MultiheadAttention,must be Tensor,N\A,"TypeError: _native_multi_head_attention(): argument 'qkv_bias' (position 7) must be Tensor, not NoneType",https://github.com/pytorch/pytorch/issues/97128,API Input Context,Modeling Parameter Space,,
,no,2023-05-31T23:02:44Z,2.0.1,cu117,Logical,Inconsistent results,torch.distributed.fsdp.FullyShardedDataParallel,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/102667,API Input Context,Modeling Parameter Space,,
,yes,2023-06-20T11:57:06Z,2.0.1,N.A,Logical,RuntimeError,torch.vmap,does not require,N\A,element 0 of tensors does not require grad and does not have a grad_fn,https://github.com/pytorch/pytorch/issues/103879,API Input Context,Modeling Parameter Space,,
,no,2023-07-14T04:38:49Z,2.0.1,N.A,Performance bug,Performance bug,torch.utils.data.DataLoader,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/105203,Test Case Context,Supporting External Library API,,
,no,2023-05-19T03:12:11Z,2.0.1,cu117,Logical,Inconsistent results,torch.utils.data.DataLoader,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101850,API Input Context,Modeling Parameter Space,,
,yes,2023-07-21T14:35:11Z,2.0.1,N.A,Logical,Inconsistent results,torch.unique,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/105742,API Input Context,Modeling Parameter Space,,
,no,2023-10-16T12:01:26Z,2.0.1,cu117,Logical,RuntimeError,torch.compile,Failed running,N\A,"torch._dynamo.exc.TorchRuntimeError: Failed running call_function <Wrapped method <original tolist>>(*(FakeTensor(..., size=(1,), dtype=torch.int64),), **{}):
.tolist() is not supported for tensor subclasses, got FakeTensor",https://github.com/pytorch/pytorch/issues/111370,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-09-08T18:02:03Z,2.0.1,cu118,Logical,Inconsistent results,torch.Tensor.any,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108882,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-07-19T17:23:18Z,2.0.1,cu117,Logical,RuntimeError,torch.sparse.sampled_addmm,but got,N\A,"RuntimeError: crow_indices is supposed to be a vector, but got 2 dimensional tensor.",https://github.com/pytorch/pytorch/issues/105548,API Input Context,Modeling Parameter Space,,
,yes,2023-09-16T20:47:59Z,2.0.1,cu117,Performance bug,Performance bug,torch.sparse.mm,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/109445,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-09-17T16:45:33Z,2.0.1,N.A,Logical,Inconsistent results,torch.sparse_coo_tensor,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109462,API Input Context,Modeling Parameter Space,,
,no,2023-06-21T06:25:05Z,2.0.1,cu118,Logical,RuntimeError,torch.save,Parent directory,N\A,RuntimeError: Parent directory C:\Users\ユーザー名\source\ does not exist.,https://github.com/pytorch/pytorch/issues/103949,API Input Context,Modeling Parameter Space,,
,no,2023-05-16T23:58:51Z,2.0.1,cu117,Logical,RuntimeError,torch.profiler.profile,INTERNAL ASSERT,N\A,RuntimeError: stack.size() INTERNAL ASSERT FAILED at,https://github.com/pytorch/pytorch/issues/101632,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-07-27T08:09:57Z,2.0.1,N.A,Logical,TypeError,torch.optim.Optimizer.add_param_group,invalid combination of arguments,N\A,"TypeError: _foreach_addcdiv_() received an invalid combination of arguments - got (list, list, tuple, list), but expected one of:",https://github.com/pytorch/pytorch/issues/106121,API Input Context,Modeling Parameter Space,,
,no,2023-08-01T03:00:21Z,2.0.1,N.A,Logical,Inconsistent results,torch.optim.lr_scheduler.StepLR,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106359,API Input Context,Modeling Parameter Space,,
,yes,2023-08-22T15:43:54Z,2.0.1,N.A,Logical,RuntimeError,torch.ops.aten.clamp.out,value cannot be converted,N\A,# RuntimeError: value cannot be converted to type int8_t without overflow,https://github.com/pytorch/pytorch/issues/107702,API Input Context,Modeling Parameter Space,,
,yes,2023-08-22T15:35:09Z,2.0.1,N.A,Logical,Inconsistent results,torch.ops.aten.arange.out,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107701,API Input Context,Modeling Specific Data Types,,
,yes,2023-09-26T04:47:26Z,2.0.1,N.A,Vulnerability,Floating point exception,torch.onnx.export,Floating point exception,N\A,Floating point exception (core dumped),https://github.com/pytorch/pytorch/issues/110056,API Input Context,Empty Input Tensor,,
,yes,2023-09-16T04:08:35Z,2.0.1,cu117,Logical,ValueError,torch.onnx.export,symbolic expected,N\A,"torch.onnx.errors.SymbolicValueError: ONNX symbolic expected the output of `%permute : Tensor(*, *, *) = onnx::Transpose[perm=[0, 2, 1]](%quantize_per_tensor), scope: torch.fx.graph_module.GraphModule:: # <eval_with_key>.6:8:14",https://github.com/pytorch/pytorch/issues/109425,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-07-31T09:19:44Z,2.0.1,cu117,Logical,Inconsistent results,torch.onnx.export,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106298,API Input Context,Modeling Parameter Space,,
,yes,2023-07-18T08:59:37Z,2.0.1,cu117,Logical,ValueError,torch.onnx.export,export of convolution,N\A,"torch.onnx.errors.SymbolicValueError: Unsupported: ONNX export of convolution for kernel of unknown shape.  [Caused by the value '0 defined in (%0 : Float(1, 3, 10, 10, strides=[300, 100, 10, 1], requires_grad=0, device=cpu) = prim::Param()",https://github.com/pytorch/pytorch/issues/105454,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-06-29T10:05:56Z,2.0.1,cu117,Logical,Others,torch.onnx.export,ONNXRuntimeError,N\A,onnxruntime.capi.onnxruntime_pybind11_state.InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : ,https://github.com/pytorch/pytorch/issues/104403,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-09-29T15:52:26Z,2.0.1,N.A,Logical,TypeError,torch.ones,unsupported ScalarType,N\A,TypeError: Got unsupported ScalarType BFloat16,https://github.com/pytorch/pytorch/issues/110285,API Input Context,Modeling Specific Data Types,,
,yes,2023-06-01T15:24:36Z,2.0.1,N.A,Logical,Inconsistent results,torch.nn.utils.parametrizations.orthogonal,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/102740,API Input Context,Modeling Specific Data Types,,
,no,2023-07-20T02:41:58Z,2.0.1,cu117,Vulnerability,Others,torch.nn.TransformerDecoderLayer,integer division,N\A,ZeroDivisionError: integer division or modulo by zero,https://github.com/pytorch/pytorch/issues/105632,API Input Context,Zero Integer Argument,,
,yes,2023-09-13T10:27:51Z,2.0.1,cu118,Logical,Inconsistent results,torch.nn.parallel.DistributedDataParallel,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109191,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-08-14T09:18:31Z,2.0.1,N.A,Logical,Inconsistent results,torch.nn.MaxPool2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107125,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-08-05T12:53:17Z,2.0.1,cu117,Logical,Inconsistent results,torch.nn.Linear,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106665,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-05-15T17:57:33Z,2.0.1,cu117,Performance bug,Performance bug,torch.nn.Sequential,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/101428,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-07-17T08:38:49Z,2.0.1,cu118,Logical,RuntimeError,torch.nn.LeakyReLU,INTERNAL ASSERT,N\A,"RuntimeError: 0 INTERNAL ASSERT FAILED at ""../torch/csrc/jit/ir/alias_analysis.cpp"":615, please report a bug to PyTorch. We don't have an op for aten::leaky_relu but it isn't a special case.  Argument types: Tensor, bool, ",https://github.com/pytorch/pytorch/issues/105313,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-08-30T10:24:22Z,2.0.1,cu117,Logical,TypeError,torch.nn.functional.pad,Expected a value,N\A,"aten::pad(Tensor self, SymInt[] pad, str mode=""constant"", float? value=None) -> Tensor: Expected a value of type 'Optional[float]' for argument 'value' but instead found type 'bool'.",https://github.com/pytorch/pytorch/issues/108226,API Input Context,Modeling Specific Data Types,,
FreeFuzz,yes,2023-08-31T11:06:47Z,2.0.1,N.A,Logical,AssertionError,torch.nn.functional.cross_entropy,Assertion,N\A,"../aten/src/ATen/native/cuda/NLLLoss2d.cu:107: nll_loss2d_forward_kernel: block: [188,0,0], thread: [32,0,0] Assertion input_index >= 0failed. ../aten/src/ATen/native/cuda NLLLoss2d.cu:107: nll_loss2d_forward_kernel: ",https://github.com/pytorch/pytorch/issues/108345,API Input Context,Modeling Parameter Space,,
,yes,2023-08-22T12:04:15Z,2.0.1,CPU,Logical,Inconsistent results,torch.nn.functional.cross_entropy,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107680,API Input Context,Modeling Parameter Space,,
,yes,2023-07-04T19:02:39Z,2.0.1,N.A,Logical,RuntimeError,torch.nn.functional.conv1d,please report a bug to PyTorch,N\A,"RuntimeError: isList() INTERNAL ASSERT FAILED at ""/Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/core/ivalue_inl.h"":2034, please report a bug to PyTorch. Expected GenericList but got String",https://github.com/pytorch/pytorch/issues/104606,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-09-17T06:11:15Z,2.0.1,N.A,Logical,Inconsistent results,torch.nn.ConvTranspose2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109453,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-07-06T12:17:30Z,2.0.1,cu117,Performance bug,Performance bug,torch.nn.Conv3d,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/104701,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-10-02T13:34:23Z,2.0.1,N.A,Logical,Inconsistent results,torch.nn.Conv2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110387,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,no,2023-09-15T16:31:52Z,2.0.1,N.A,Logical,RuntimeError,torch.distributed.fsdp.FullyShardedDataParallel,view of its base has been modified inplace,N\A,RuntimeError: Output 0 of ViewBackward0 is a view and its base or another view of its base has been modified inplace. This view is the output of a function that returns multiple views. Such functions do not allow the output views to be modified inplace. You should replace the inplace operation by an out-of-place one.,https://github.com/pytorch/pytorch/issues/109385,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-10T07:06:15Z,2.0.1,cu117,Logical,RuntimeError,torch.nn.Conv2d,Invalid padding size,N\A,"RuntimeError: Invalid padding size, expected 2 but got 4",https://github.com/pytorch/pytorch/issues/104860,API Input Context,Modeling Parameter Space,,
,yes,2023-05-16T07:34:32Z,2.0.1,cu118,Performance bug,Performance bug,torch.nn.Conv2d,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/101529,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-08-28T07:45:45Z,2.0.1,cu118,Logical,RuntimeError,torch.nn.ConstantPad2d,Trying to create tensor,N\A,RuntimeError: Trying to create tensor with negative dimension -1: [-1],https://github.com/pytorch/pytorch/issues/108043,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
DeepRel,yes,2023-07-20T02:38:55Z,2.0.1,cu117,Logical,RuntimeError,torch.nn.ConstantPad2d,must be non-negative,N\A,RuntimeError: narrow(): length must be non-negative.,https://github.com/pytorch/pytorch/issues/105629,API Input Context,Negative Integer Argument,,
,yes,2023-07-20T02:38:55Z,2.0.1,cu117,Logical,RuntimeError,torch.nn.ConstantPad2d,must be non-negative,N\A,RuntimeError: narrow(): length must be non-negative.,https://github.com/pytorch/pytorch/issues/105629,API Input Context,Zero Integer Argument,,
,yes,2023-10-04T14:38:10Z,2.0.1,cu118,Logical,RuntimeError,torch.nested.as_nested_tensor,memory format for Nested Tensors,N\A,RuntimeError: empty_like only supports contiguous memory format for Nested Tensors,https://github.com/pytorch/pytorch/issues/110516,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-09-04T15:54:12Z,2.0.1,cu117,Logical,Inconsistent results,torch.Tensor.mm,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108521,API Input Context,Modeling Parameter Space,,
,yes,2023-10-18T05:12:55Z,2.0.1,cu117,Logical,Inconsistent results,torch.matmul,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/111457,API Input Context,Modeling Specific Data Types,,
,no,2023-05-26T04:14:33Z,2.0.1,N.A,Performance bug,Performance bug,torch.load,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/102334,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-07-11T12:33:15Z,2.0.1,N.A,Logical,RuntimeError,torch.amp.autocast,INTERNAL ASSERT FAILED,N\A,"RuntimeError: isInt() INTERNAL ASSERT FAILED at ""../aten/src/ATen/core/ivalue.h"":626, please report a bug to PyTorch. ",https://github.com/pytorch/pytorch/issues/104959,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-10-16T14:31:23Z,2.0.1,cu117,Logical,Inconsistent results,torch.jit.script,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/111373,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-09-06T17:19:07Z,2.0.1,cu118,Logical,Inconsistent results,torch.jit.script,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108670,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-08-07T11:23:50Z,2.0.1,CPU,Logical,RuntimeError,torch.jit.script,undefined value torch,N\A,RuntimeError: undefined value torch:,https://github.com/pytorch/pytorch/issues/106699,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-08-20T10:27:30Z,2.0.1,CPU,Logical,Inconsistent results,torch.fx.symbolic_trace,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107538,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-09-28T21:49:11Z,2.0.1,cu118,Logical,TypeError,torch.func.functional_call,not an instance of,N\A,"raise TypeError(f""{tensor} is not an instance of torch.Tensor"")",https://github.com/pytorch/pytorch/issues/110249,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
DeepRel,yes,2023-07-10T12:19:05Z,2.0.1,cu117,Logical,Inconsistent results,torch.fft.irfftn,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/104867,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-08-08T13:32:31Z,2.0.1,cu118,Logical,Inconsistent results,torch.fake_quantize_per_tensor_affine,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106780,API Input Context,Modeling Parameter Space,,
,yes,2023-09-26T19:56:04Z,2.0.1,N.A,Logical,AssertionError,torch.onnx.export,doesn't match mode,N\A,AssertionError: fake mode (<torch._subclasses.fake_tensor.FakeTensorMode object at 0x7fe4259ac760>) from active fake mode 0 doesn't match mode (<torch._subclasses.fake_tensor.FakeTensorMode object at 0x7fe355ae0760>) from fake tensor input 0,https://github.com/pytorch/pytorch/issues/110100,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-06-26T07:44:44Z,2.0.1,cu117,Performance bug,Performance bug,torch.distributed.scatter,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/104174,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-08-17T03:02:53Z,2.0.1,cu117,Logical,Inconsistent results,torch.distributed.dist.scatter,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107352,API Input Context,Inputs with Incompatible Shapes,,
,no,2023-08-30T03:12:34Z,2.0.1,cu118,Logical,Others,torch.distributed.all_reduce,exception not rethrown,N\A,FATAL: exception not rethrown,https://github.com/pytorch/pytorch/issues/108210,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-09-03T17:42:17Z,2.0.1,cu117,Performance bug,Performance bug,torch.Tensor.cummax,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/108496,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-10-06T18:31:54Z,2.0.1,N.A,Logical,RuntimeError,torch.compile,Failed running,N\A,"RuntimeError: Failed running call_function <built-in function add>(*(s0 > 1, s0 < 3), **{}):",https://github.com/pytorch/pytorch/issues/110738,API Input Context,Modeling Parameter Space,,
,no,2023-09-14T18:36:01Z,2.0.1,N.A,Logical,Unsupported Error,torch.compile,GenericContextWrappingVariable,N\A,torch._dynamo.exc.Unsupported: call_method GenericContextWrappingVariable() enter_context [GenericContextWrappingVariable()] {},https://github.com/pytorch/pytorch/issues/109309,Test Case Context,Supporting External Library API,,
,yes,2023-11-15T12:35:57Z,2.1.0,,Logical,Others,torch.nn.Upsample,Failed to export the model,N\A,"torch.onnx.OnnxExporterError: Failed to export the model to ONNX. Generating SARIF report at 'report_dynamo_export.sarif'. SARIF is a standard format for the output of static analysis tools. SARIF logs can be loaded in VS Code SARIF viewer extension, or SARIF web viewer (https://microsoft.github.io/sarif-web-component/). Please report a bug on PyTorch Github: https://github.com/pytorch/pytorch/issues",https://github.com/pytorch/pytorch/issues/113755,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-08-15T14:24:15Z,2.0.1,cu117,Logical,Unsupported Error,torch.compile,inline in skipfiles,N\A,Unsupported: inline in skipfiles: _SpecialForm.__getitem__  | inner /opt/conda/lib/python3.10/typing.py,https://github.com/pytorch/pytorch/issues/107232,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-08-14T14:16:25Z,2.0.1,CPU,Logical,AttributeError,torch.compile,has no attribute,N\A,AttributeError: module '__main__' has no attribute 'MySubClass',https://github.com/pytorch/pytorch/issues/107143,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-08-12T02:59:47Z,2.0.1,N.A,Logical,Graph break,torch.compile,Graph break,N\A,Graph break,https://github.com/pytorch/pytorch/issues/107076,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-21T22:03:11Z,2.0.1,N.A,Performance bug,Performance bug,torch.compile,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/105768,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-07-13T21:45:35Z,2.0.1,cu117,Performance bug,Performance bug,torch.compile,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/105181,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-06-23T08:08:38Z,2.0.1,cu117,Performance bug,Performance bug,torch.nn.Sequential,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/104095,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-07-02T06:25:03Z,2.0.1,cu118,Logical,AttributeError,torch.Tensor.as_strided_,raised AttributeError,N\A,BackendCompilerFailed: debug_wrapper raised AttributeError: The underlying op of 'aten.sym_storage_offset' has no overload name 'name',https://github.com/pytorch/pytorch/issues/104505,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-10-04T14:35:59Z,2.0.1,cu117,Logical,Inconsistent results,torch.ao.quantization.quantize_dynamic,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110515,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,no,2023-09-06T06:21:49Z,2.0.1,cu117,Logical,Inconsistent results,torch.amp.autocast,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108627,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-10-02T23:53:57Z,2.0.1,N.A,Logical,RuntimeError,torch.func.jacrev,Cannot access data pointer,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/110422,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2023-06-08T17:05:25Z,2.1.0,cu118,Performance bug,Performance bug,torch.nn.parallel.DistributedDataParallel,Performance bug,N\A,High PCIe traffic,https://github.com/pytorch/pytorch/issues/103254,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-20T19:17:12Z,2.1.0,cu118,Logical,RuntimeError,torch.nn.parallel.DistributedDataParallel,Input Tensor data type,N\A,RuntimeError: Input Tensor data type is not supported for NCCL process group: ComplexFloat,https://github.com/pytorch/pytorch/issues/99652,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-03-24T14:47:03Z,2.1.0,cu118,Logical,Others,torch.nn.TransformerEncoderLayer,Converting mask without,N\A,UserWarning,https://github.com/pytorch/pytorch/issues/97532,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-08-17T16:53:23Z,2.1.0,CPU,Logical,Others,torch.utils.data.random_split,has incompatible type,N\A,"error: Argument 1 to ""random_split"" has incompatible type ""range""; expected ""Dataset[<nothing>]""",https://github.com/pytorch/pytorch/issues/107392,API Input Context,Modeling Parameter Space,,
,yes,2023-10-13T14:05:58Z,2.1.0,cu118,Logical,AttributeError,torch.utils.checkpoint.checkpoint,has no attribute,N\A,AttributeError: 'Tensor' object has no attribute 'huhu',https://github.com/pytorch/pytorch/issues/111206,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-10-18T21:16:16Z,2.1.0,N.A,Logical,RuntimeError,torch.Tensor.sum,operands have incompatible,N\A,"RuntimeError: sparse_mask(): operands have incompatible sizes; self has size [2449029, 1] but mask has size [2449029, 2449029]",https://github.com/pytorch/pytorch/issues/111509,API Input Context,Modeling Parameter Space,,
,yes,2023-10-07T03:30:06Z,2.1.0,N.A,Logical,Inconsistent results,torch.Tensor.set_,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110780,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-08-21T14:03:25Z,2.1.0,cu118,Logical,RuntimeError,torch.Tensor.repeat_interleave,Expected all tensors to be on the same device,N\A,"RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu! (when checking argument for argument index in method wrapper_CUDA__index_select)",https://github.com/pytorch/pytorch/issues/107591,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-04-05T16:34:29Z,2.1.0,cu118,Logical,Inconsistent results,torch.Tensor.flip,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/98422,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-10-19T01:19:27Z,2.1.0,N.A,Logical,ValueError,torch.Tensor.dims,is unbound,N\A,ValueError: dimension d0 is unbound,https://github.com/pytorch/pytorch/issues/111525,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-10-24T15:37:51Z,2.1.0,,Logical,Others,torch.logical_and,An output with one or more elements,N\A,"UserWarning: An output with one or more elements was resized since it had shape [2, 128], which does not match the required output shape [4, 2, 128]. This behavior is deprecated, and in a future PyTorch release outputs will not be resized unless they have zero elements. You can explicitly reuse an out tensor t by resizing it, inplace, to zero elements with t.resize_(0). (Triggered internally at /opt/conda/conda-bld/pytorch_1695392022560/work/aten/src/ATen/native/Resize.cpp:28.)
  torch.logical_and(b, a)",https://github.com/pytorch/pytorch/issues/111925,API Input Context,Inputs with Incompatible Shapes,,
,yes,2023-09-12T13:27:01Z,2.1.0,CPU,Logical,TypeError,torch.sparse_coo_tensor,only integer tensors of a single element,N\A,# TypeError: only integer tensors of a single element can be converted to an index,https://github.com/pytorch/pytorch/issues/109108,API Input Context,Modeling Parameter Space,,
,yes,2023-07-24T03:52:52Z,2.1.0,N.A,Logical,Inconsistent results,torch.Tensor.scatter,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/105822,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-08-12T16:21:00Z,2.1.0,N.A,Logical,RuntimeError,torch.randn,not implemented for,N\A,"RuntimeError: ""add_stub"" not implemented for 'Float8_e5m2'",https://github.com/pytorch/pytorch/issues/107087,API Input Context,Modeling Specific Data Types,,
,yes,2023-10-10T11:34:54Z,2.1.0,N.A,Logical,RuntimeError,torch.optim,Tensors of the same index must be on the same device,N\A,RuntimeError: Tensors of the same index must be on the same device and the same dtype except `step` tensors that can be CPU and float32 notwithstanding,https://github.com/pytorch/pytorch/issues/110940,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,no,2023-05-31T11:06:47Z,2.1.0,CPU,Logical,RuntimeError,torch.ops.prims.sum,takes 2 positional argument(s),N\A,"RuntimeError: prims::sum() takes 2 positional argument(s) but 3 was/were given.  Declaration: prims::sum(Tensor inp, int[]? dims, *, ScalarType? output_dtype=None) -> Tensor",https://github.com/pytorch/pytorch/issues/102610,API Input Context,Modeling Specific Data Types,,
,no,2023-03-22T08:33:52Z,2.1.0,CPU,Logical,Inconsistent results,torch.ops.aten.pow,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/97329,API Input Context,Modeling Specific Data Types,,
,yes,2023-05-09T16:44:54Z,2.1.0,CPU,Logical,Inconsistent results,torch.ops.aten.native_batch_norm,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/100985,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-05-31T10:58:54Z,2.1.0,CPU,Logical,RuntimeError,torch.ops.aten.le,Expected a value of type,N\A,"RuntimeError: prims::le(Tensor self, Tensor other) -> Tensor: Expected a value of type 'Tensor' for argument 'other' but instead found type 'float'.",https://github.com/pytorch/pytorch/issues/102609,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-09-01T09:19:57Z,2.1.0,,Logical,Inconsistent results,torch.einsum,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108407,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-09-27T12:54:55Z,2.1.0,N.A,Logical,ValueError,torch.onnx.export,When exporting a ScriptModule or ScriptFunction,N\A,"ValueError: args contained 2 None's after flattening. When exporting a ScriptModule or ScriptFunction, no args may be None because that breaks type propagation.",https://github.com/pytorch/pytorch/issues/110154,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-08-22T12:58:26Z,2.1.0,cu121,Logical,Others,torch.onnx.export,The exported ONNX model failed,N\A,"/usr/local/lib/python3.10/dist-packages/torch/onnx/utils.py:1672: UserWarning: The exported ONNX model failed ONNX shape inference. The model will not be executable by the ONNX Runtime. If this is unintended and you believe there is a bug, please report an issue at https://github.com/pytorch/pytorch/issues. Error reported by strict ONNX shape inference: [ShapeInferenceError] Shape inference error(s): (op_type:Split, node name: /Split): [ShapeInferenceError] Neither 'split' input nor 'num_outputs' attribute has been given
(op_type:Sigmoid, node name: /Sigmoid): [TypeInferenceError] Input 0 expected to have type but instead is null
(op_type:Mul, node name: /Mul): [TypeInferenceError] Input 0 expected to have type but instead is null
 (Triggered internally at ../torch/csrc/jit/serialization/export.cpp:1403.)
  _C._check_onnx_proto(proto)",https://github.com/pytorch/pytorch/issues/107685,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-08-19T20:07:50Z,2.1.0,cu121,Logical,Unsupported Error,torch.onnx.export,Exporting the operator,N\A,UnsupportedOperatorError: Exporting the operator 'quantized::linear_relu' to ONNX opset version 16 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues.,https://github.com/pytorch/pytorch/issues/107522,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-26T05:38:37Z,2.1.0,,Logical,RuntimeError,torch.nn.MultiLabelMarginLoss,Expected non-empty vector or matrix with,N\A,"RuntimeError: Expected non-empty vector or matrix with optional 0-dim batch size, but got: [20, 9, 20]",https://github.com/pytorch/pytorch/issues/106011,API Input Context,Inputs with Incompatible Shapes,,
,no,2023-10-19T01:40:18Z,2.1.0,N.A,Logical,Inconsistent results,torch.no_grad,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/111528,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-08-22T05:46:29Z,2.1.0,CPU,Logical,RuntimeError,torch.nn.TransformerEncoder,expected scalar type,N\A,RuntimeError: expected scalar type BFloat16 but found Float,https://github.com/pytorch/pytorch/issues/107663,API Input Context,Modeling Parameter Space,,
,yes,2023-05-10T01:24:10Z,2.1.0,cu118,Logical,Inconsistent results,torch.nn.Softmax,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101039,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-07-25T06:57:17Z,2.1.0,N.A,Logical,Inconsistent results,torch.nn.MaxPool2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/105901,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-05-22T08:19:33Z,2.1.0,N.A,Logical,Inconsistent results,torch.nn.functional.scaled_dot_product_attention,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101967,API Input Context,Modeling Parameter Space,,
,yes,2023-03-28T17:52:04Z,2.1.0,cu116,Logical,ValueError,torch.nn.functional.pad,Mask argument cannot be block type,N\A,ValueError('Mask argument cannot be block type if pointer argument is not a block'),https://github.com/pytorch/pytorch/issues/97807,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-26T02:17:10Z,2.1.0,N.A,Logical,BackendCompilerFailed,torch.nn.functional.linear,is not tracked with proxy for,N\A,RuntimeError: s0 is not tracked with proxy for <torch.fx.experimental.proxy_tensor.PythonKeyTracer object at 0x7f25897eed7,https://github.com/pytorch/pytorch/issues/100055,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-10-18T16:54:10Z,2.1.0,cu121,Logical,Inconsistent results,torch.nn.CrossEntropyLoss,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/111484,API Input Context,Modeling Parameter Space,,
,yes,2023-10-09T19:29:22Z,2.1.0,cu121,Performance bug,Performance bug,torch.nn.Conv3d,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/110885,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-06-19T15:26:56Z,2.1.0,cu121,Vulnerability,Segmentation fault,torch.nn.Conv2d,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/103852,Test Case Context,Others,,
,yes,2023-05-14T08:31:07Z,2.1.0,N.A,Logical,AssertionError,torch.nn.Conv2d,expected size,N\A,"AssertionError: expected size 3==3, stride 98==784 at dim=0",https://github.com/pytorch/pytorch/issues/101356,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2023-09-18T22:49:15Z,2.1.0,N.A,Logical,RuntimeError,torch.nn.Conv1d,Failed running,N\A,"torch._dynamo.exc.TorchRuntimeError: Failed running call_module L__self___conv(*(FakeTensor(..., size=(1, 6, 3)),), **{}): Trying to create tensor with negative dimension -2: [1, 6, -2]",https://github.com/pytorch/pytorch/issues/109552,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
TitanFuzz,yes,2023-07-08T09:00:01Z,2.1.0,cu118,Logical,RuntimeError,torch.nn.AdaptiveAvgPool3d,Input dimension should be at least,N\A,"With output_size = [1,2,3,4], the error is: ""ValueError: Input dimension should be at least 5"".
With output_size = [1,2,3,4,5], the error is: ""ValueError: Input dimension should be at least 6"".
With output_size = [1,2,3,4,5,6], the error is: ""ValueError: Input dimension should be at least 7"".
Additionally, when the output_size is less than 3, the error message raised is: ""RuntimeError: adaptive_avg_pool3d: output_size must be 3"". this error message should be raised too when the output_size is bigger than 3?",https://github.com/pytorch/pytorch/issues/104823,API Input Context,Inputs with Incompatible Shapes,,
,yes,2023-07-06T10:31:43Z,2.1.0,cu118,Logical,RuntimeError,torch.nn.AdaptiveAvgPool2d,should be at least,N\A,Input dimension should be at least 5,https://github.com/pytorch/pytorch/issues/104698,API Input Context,Inputs with Incompatible Shapes,,
,yes,2023-04-01T04:36:37Z,2.1.0,cu117,Logical,RuntimeError,torch.Tensor.movedim,out of bounds for,N\A,"RuntimeError: setStorage: sizes [2, 2, 6, 3, 2], strides [72, 36, 6, 2, 1], storage offset 0, and itemsize 4 requiring a storage size of 576 are out of bounds for storage of size 288",https://github.com/pytorch/pytorch/issues/98122,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-09-19T08:21:03Z,2.1.0,CPU,Logical,Inconsistent results,torch.max_pool2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109586,API Input Context,Negative Integer Argument,,
,yes,2023-09-18T13:41:02Z,2.1.0,N.A,Logical,Inconsistent results,torch.lobpcg,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/109497,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-10-11T13:13:54Z,2.1.0,cu121,Logical,Others,torch.linalg.matrix_norm,Expected a value,N\A,"Expected a value of type 'str' for argument 'ord' but instead found type 'Union[int, str]'",https://github.com/pytorch/pytorch/issues/111029,API Input Context,Modeling Parameter Space,,
,no,2023-08-17T16:17:24Z,2.1.0,CPU,Logical,Others,torch.library.Library.define,cannot be assigned,N\A,"error: Expression of type ""_DispatchModule"" cannot be assigned to declared type ""str""",https://github.com/pytorch/pytorch/issues/107387,API Input Context,Modeling Parameter Space,,
,yes,2023-08-19T02:37:47Z,2.1.0,cu118,Logical,RuntimeError,torch.jit.trace,expected end of number,N\A,RuntimeError: default_program(22): error: extra text after expected end of number,https://github.com/pytorch/pytorch/issues/107503,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-06T20:08:35Z,2.1.0,CPU,Logical,RuntimeError,torch.jit.trace,Arguments for call are not valid,N\A,RuntimeError:  Arguments for call are not valid. The following variants are available:,https://github.com/pytorch/pytorch/issues/104732,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-10-18T16:11:12Z,2.1.0,cpu,Logical,Inconsistent results,torch.jit.script,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/111480,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-06-13T00:38:57Z,2.1.0,cu118,Logical,Others,torch.fx.experimental.proxy_tensor.make_fx,Failed running,N\A,"Failed running call_function <function interpolate at 0x7f1b7d1769d0>(*(FakeTensor(..., device='cuda:0', size=(2, 3, 4, 5)),), **{'size': (10, 20, 30), 'mode': 'trilinear', 'align_corners': True}): Input and output must have the same number of spatial dimensions, but got input with spatial dimensions of [4, 5] and output size of (10, 20, 30). Please provide input tensor in (N, C, d1, d2, ...,dK) format and output size in (o1, o2, ...,oK) format.",https://github.com/pytorch/pytorch/issues/103473,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-03-22T09:58:49Z,2.1.0,cu117,Logical,Inconsistent results,torch.Tensor.fmod,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/97333,Test Case Context,Modeling Compile/Eager Execution Mode,,
,no,2023-05-21T12:55:42Z,2.1.0,N.A,Logical,Inconsistent results,torch.Tensor.flip,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101950,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-10-13T22:10:35Z,2.1.0,cu118,Logical,Others,torch.Tensor.equal,Crash,N\A,Crash,https://github.com/pytorch/pytorch/issues/111251,API Input Context,Modeling Parameter Space,,
,yes,2022-12-12T14:43:26Z,2.1.0,CPU,Logical,Inconsistent results,torch.empty,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/90695,API Input Context,Modeling Parameter Space,1,trigger: output of torch.empty seems wrong with sparse arguments layout=sparse_csr|sparse_csc
,yes,2023-03-18T11:01:11Z,2.1.0,cu117,Vulnerability,Segmentation fault,torch.embedding,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/97088,API Input Context,Large Input Tensor,,
,yes,2023-08-23T21:36:32Z,2.1.0,N.A,Logical,Inconsistent results,torch.distributions.Pareto.sample,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107821,API Input Context,Modeling Parameter Space,,
,yes,2023-08-31T22:00:54Z,2.1.0,N.A,Performance bug,Performance bug,torch.distributed.isend,Hang,N\A,Hang,https://github.com/pytorch/pytorch/issues/108378,Test Case Context,Oracle for Performance Bug Detection,,
,no,2023-05-16T09:37:33Z,2.1.0,N.A,Logical,RuntimeError,torch.cuda.set_device,Expected one of cpu,N\A,"# RuntimeError: Expected one of cpu, cuda, ipu, xpu, mkldnn, opengl, opencl, ideep, hip, ve, fpga, ort, xla, lazy, vulkan, mps, meta, hpu, mtia, privateuseone device type at start of device string: gpu",https://github.com/pytorch/pytorch/issues/101536,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-09-06T17:45:45Z,2.0.0,,Logical,RuntimeError,torch.nn.functional.conv_transpose2d,could not construct a memory,N\A,RuntimeError: could not construct a memory descriptor using a format tag,https://github.com/pytorch/pytorch/issues/108676,API Input Context,Modeling Parameter Space,,
,yes,2023-10-13T11:55:38Z,2.1.0,N.A,Logical,RuntimeError,torch.compile,Failed running,N\A,torch._dynamo.exc.TorchRuntimeError: Failed running call_function <built-in method sum of type object at 0x7f61e0ce2220>(*(FakeTensor(,https://github.com/pytorch/pytorch/issues/111198,API Input Context,Modeling Parameter Space,,
,yes,2023-10-11T10:09:20Z,2.1.0,N.A,Vulnerability,Segmentation fault,torch.compile,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/111027,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-10-07T18:38:33Z,2.1.0,N.A,Logical,RuntimeError,torch.compile,attempting to assign a,N\A,RuntimeError: attempting to assign a gradient with dtype 'float' to a tensor with dtype 'c10::Half'. Please ensure that the gradient and the tensor have the same dtype,https://github.com/pytorch/pytorch/issues/110797,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-09-06T09:45:04Z,2.0.1,,Logical,Inconsistent results,torch.Tensor.topk,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108642,API Input Context,Modeling Parameter Space,,
,yes,2023-09-01T17:53:14Z,2.1.0,N.A,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108432,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-08-23T22:08:03Z,2.1.0,N.A,Logical,Others,torch.compile,Please convert all,N\A,"torch._dynamo.exc.InternalTorchDynamoError: Please convert all Tensors to FakeTensors first or instantiate FakeTensorMode with 'allow_non_fake_inputs'. Found in aten.copy_.default(tensor([...], size=(50, 100)), FakeTensor(..., size=(50, 100)))",https://github.com/pytorch/pytorch/issues/107824,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,yes,2023-08-23T17:56:40Z,2.1.0,cu121,Logical,RuntimeError,torch.compile,Failed running,N\A,"torch._dynamo.exc.TorchRuntimeError: Failed running call_function <function pack_padded_sequence at 0x7ff611d61900>(*(FakeTensor(..., size=(1, 32, 32)), FakeTensor(..., size=(1,), dtype=torch.int64)), **{'batch_first': True, 'enforce_sorted': False}): 'lengths' argument should be a 1D CPU int64 tensor, but got 1D meta Long tensor",https://github.com/pytorch/pytorch/issues/107797,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-08-15T15:54:28Z,2.1.0,N.A,Logical,Graph break,torch.compile,Graph break,N\A,"[2023-08-15 09:18:38,079] torch._dynamo.symbolic_convert.__graph_breaks: [DEBUG] Graph break: call_function args: UserDefinedObjectVariable(FooTensor)  from user code at:           ",https://github.com/pytorch/pytorch/issues/107239,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-07-26T03:43:07Z,2.1.0,cu118,Logical,AssertionError,torch.compile,AssertionError,N\A,AssertionError,https://github.com/pytorch/pytorch/issues/106006,Test Case Context,Modeling Compile/Eager Execution Mode,,
,yes,2023-07-24T19:03:33Z,2.1.0,N.A,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/105863,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-07-16T14:48:37Z,2.1.0,N.A,Logical,RuntimeError,torch.compile,Expected a value of type,N\A,Torch.compile Error: RuntimeError: aten::_conj() Expected a value of type 'Tensor' for argument 'self' but instead found type 'complex',https://github.com/pytorch/pytorch/issues/105290,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-07-11T05:58:55Z,2.1.0,cu118,Logical,RuntimeError,torch.compile,one of the variables needed for gradient,N\A,"RuntimeError : one of the variables needed for gradient computation has been modified by an inplace operation: [torch.FloatTensor [16, 16]]",https://github.com/pytorch/pytorch/issues/104943,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-04T21:26:11Z,2.1.0,cu121,Logical,RuntimeError,torch.compile,INTERNAL ASSERT FAILED,N\A,"torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised: RuntimeError: list_trace[idx] == nullptr INTERNAL ASSERT FAILED at ""../torch/csrc/jit/frontend/tracer.cpp"":1016, please report a bug to PyTorch.",https://github.com/pytorch/pytorch/issues/104610,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-06-22T04:44:24Z,2.1.0,cu118,Logical,AssertionError,torch.nn.functional.linear,The values for attribute,N\A,"AssertionError: The values for attribute 'shape' do not match: torch.Size([1, 1, 2]) != torch.Size([1, 2, 2]).",https://github.com/pytorch/pytorch/issues/104020,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-10-19T06:55:18Z,2.1.0,cu118,Logical,Graph break,torch.compile,assert,N\A,"    assert isinstance(key, (str, int))
AssertionError: ",https://github.com/pytorch/pytorch/issues/111551,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-06-06T18:47:59Z,2.1.0,N.A,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/103099,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-05-14T22:25:23Z,2.1.0,cu118,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101371,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2023-05-06T18:19:52Z,2.1.0,cu118,Logical,AssertionError,torch.compile,The values for attribute,N\A,"AssertionError: The values for attribute 'shape' do not match: torch.Size([2, 9, 16, 16]) != torch.Size([2, 6, 16, 16]).",https://github.com/pytorch/pytorch/issues/100807,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-05-06T00:04:43Z,2.1.0,cu118,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/100775,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-05-05T18:46:03Z,2.1.0,cu117,Logical,RuntimeError,torch.compile,Failed running,N\A,"RuntimeError: Failed running call_method forward(*(Linear(in_features=5, out_features=5, bias=True), FakeTensor(..., size=(1, 5))), **{}):",https://github.com/pytorch/pytorch/issues/100733,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-30T18:44:23Z,2.1.0,cu118,Logical,IndexError,torch.sum,out of bounds for,N\A,IndexError: index 3 is out of bounds for dimension 0 with size 2,https://github.com/pytorch/pytorch/issues/100343,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-29T16:28:59Z,2.1.0,cu118,Logical,RuntimeError,torch.compile,Expected size,N\A,"RuntimeError: Expected size for first two dimensions of batch2 tensor to be: [4, 1] but got: [4, 4].",https://github.com/pytorch/pytorch/issues/100316,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-24T13:30:37Z,2.1.0,cu118,Logical,TypeError,torch.compile,unsupported operand type,N\A,TypeError: unsupported operand type(s) for @: 'Tensor' and 'Tensor',https://github.com/pytorch/pytorch/issues/99874,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-04-21T02:54:49Z,2.1.0,cu118,Logical,TypeError,torch.compile,multiply sequence by,N\A,TypeError: can't multiply sequence by non-int of type 'float',https://github.com/pytorch/pytorch/issues/99689,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-04-20T18:49:06Z,2.1.0,cu118,Logical,RuntimeError,torch.compile,The expanded size of the tensor,N\A,"RuntimeError: The expanded size of the tensor (4) must match the existing size (2) at non-singleton dimension 0.  Target sizes: [4, 4].  Tensor sizes: [2, 4]",https://github.com/pytorch/pytorch/issues/99649,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-04-12T21:36:26Z,2.1.0,cu118,Logical,RuntimeError,torch.compile,expected scalar type Long,N\A,RuntimeError: expected scalar type Long but found Float,https://github.com/pytorch/pytorch/issues/98978,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
TitanFuzz,yes,2023-04-12T20:45:41Z,2.1.0,cu118,Logical,RuntimeError,torch.nn.functional.dropout,result type Float can't be,N\A,RuntimeError: result type Float can't be cast to the desired output type Long,https://github.com/pytorch/pytorch/issues/98970,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-03-20T06:25:37Z,2.1.0,cu116,Logical,Inconsistent results,torch.compile,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/97130,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-10-19T11:46:56Z,2.1.0,N.A,Performance bug,Performance bug,torch.autograd.grad,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/111563,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-10-14T03:41:19Z,2.1.0,cu121,Logical,RuntimeError,torch.autocast,Expected attn_mask dtype to be bool,N\A,"RuntimeError: Expected attn_mask dtype to be bool or to match query dtype, but got attn_mask.dtype: float and  query.dtype: c10::BFloat16 instead.",https://github.com/pytorch/pytorch/issues/111276,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-10-18T03:46:51Z,2.1.0,cu121,Performance bug,Performance bug,torch.autocast,Hang,N\A,Hang,https://github.com/pytorch/pytorch/issues/111456,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-03-15T00:30:22Z,2.0.0,,Logical,RuntimeError,torch.Tensor.cumprod,INTERNAL ASSERT FAILED,N\A,RuntimeError: t == DeviceType::CUDA INTERNAL ASSERT FAILED ,https://github.com/pytorch/pytorch/issues/96789,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2023-04-26T01:53:22Z,2.1.0,N.A,Logical,Others,torch.ao.quantization.observer.HistogramObserver.with_args,cannot convert float infinity to integer,N\A,OverflowError: cannot convert float infinity to integer,https://github.com/pytorch/pytorch/issues/100051,API Input Context,Modeling Parameter Space,,
,yes,2023-03-14T20:15:43Z,2.0.0,,Logical,RuntimeError,torch.sparse.mm,INTERNAL ASSERT FAILED,N\A, RuntimeError: mat2_.is_sparse() INTERNAL ASSERT FAILED at ,https://github.com/pytorch/pytorch/issues/96769,Test Case Context,Modeling Multiple API Usage Pattern,,
ACETest,yes,2023-04-01T08:58:55Z,2.1.0,cu118,Vulnerability,Segmentation fault,torch._grid_sampler_2d_cpu_fallback,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/98136,API Input Context,Negative Integer Argument,,
,yes,2023-03-29T04:18:46Z,2.1.0,cu116,Logical,RuntimeError,torch._enable_functionalization,on tensor with symbolic,N\A,RuntimeError: Cannot call storage_offset() on tensor with symbolic sizes/strides  ,https://github.com/pytorch/pytorch/issues/97852,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,no,2023-01-27T13:49:57Z,2.1.0,N.A,Logical,Unsupported Error,torch._dynamo.export,BuiltinVariable(dict),N\A,torch._dynamo.exc.Unsupported: call_function BuiltinVariable(dict) [ListVariable()] {},https://github.com/pytorch/pytorch/issues/93152,API Input Context,Modeling Parameter Space,,
,no,2023-01-27T13:48:19Z,2.1.0,N.A,Logical,Unsupported Error,torch._dynamo.export,in skip_files,N\A,torch._dynamo.exc.Unsupported: call_function in skip_files,https://github.com/pytorch/pytorch/issues/93151,API Input Context,Modeling Parameter Space,,
,no,2023-10-15T14:55:13Z,2.1.0,cu121,Performance bug,Performance bug,torch._C.PyTorchFileReader,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/issues/111330,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-08-03T23:23:15Z,2.1.0,N.A,Logical,Inconsistent results,torch.Tensor.expand,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106584,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2024-08-09T14:37:43Z,2.0.1,,Logical,Inconsistent results,torch.nn.Conv2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/133093,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2024-07-31T14:56:35Z,2.1.0,,Logical,Others,torch.nn.functional.interpolate,Arguments for call are not valid,N\A,JIT SCRIPT ERROR: Arguments for call are not valid. The following variants are available:,https://github.com/pytorch/pytorch/issues/132240,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2024-07-20T02:42:02Z,2.0.0,,Logical,Inconsistent results,torch.nn.AvgPool2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/131272,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2024-07-04T08:44:11Z,2.0.1,,Logical,IndexError,torch.nn.GroupNorm,list index out of range,N\A,IndexError: list index out of range,https://github.com/pytorch/pytorch/issues/130108,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2024-07-04T00:34:09Z,2.0.0,,Logical,RuntimeError,torch.isin,operation not permitted when stream is capturing,N\A,RuntimeError: CUDA error: operation not permitted when stream is capturing,https://github.com/pytorch/pytorch/issues/130078,API Input Context,Large Input Tensor,,
,yes,2024-06-27T10:44:18Z,2.0.1,,Logical,Inconsistent results,torch.nn.functional.conv3d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/129649,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2024-06-21T02:51:49Z,2.1.0,,Logical,Unsupported Error,torch.onnx.dynamo_export,call_method TupleVariable(),N\A,torch._dynamo.exc.Unsupported: call_method TupleVariable() size [ConstantVariable(int)] {},https://github.com/pytorch/pytorch/issues/129200,Test Case Context,Others,,
,yes,2024-06-12T10:56:18Z,2.0.0,,Logical,ValueError,torch.nn.InstanceNorm1d,ONNX export of,N\A,"SymbolicValueError: Unsupported: ONNX export of instance_norm training for unknown batch size.  [Caused by the value 'input defined in (%input : Float(*, 3, 11, strides=[33, 11, 1], requires_grad=0, device=cpu), %running_mean : Float(3, strides=[1], requires_grad=0, device=cpu), %running_var : Float(3, strides=[1], requires_grad=0, device=cpu), %num_batches_tracked : Long(requires_grad=0, device=cpu) = prim::Param()",https://github.com/pytorch/pytorch/issues/128501,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2024-06-12T08:22:26Z,2.0.0,,Logical,TypeError,torch.meshgrid,object is not subscriptable,N\A,TypeError: 'torch._C.Value' object is not subscriptable ,https://github.com/pytorch/pytorch/issues/128494,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2024-05-27T11:06:08Z,2.1.0,,Logical,Inconsistent results,torch.linalg.lstsq,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/127223,API Input Context,Modeling Parameter Space,,
,yes,2024-05-21T07:24:37Z,2.0.0,,Logical,Inconsistent results,torch.Tensor.orgqr,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/126760,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2024-05-21T03:11:29Z,2.0.0,,Logical,Inconsistent results,torch.Tensor.orgqr,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/126748,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2024-03-11T23:20:03Z,2.1.0,,Logical,RuntimeError,torch.linalg.ldl_factor,INTERNAL ASSERT FAILED,N\A,"RuntimeError: false INTERNAL ASSERT FAILED at ""../aten/src/ATen/native/BatchLinearAlgebra.cpp"":1568, please report a bug to PyTorch. torch.linalg.ldl_factor: Unknown error code: 1.",https://github.com/pytorch/pytorch/issues/121683,API Input Context,Zero Input Tensor,,
,yes,2024-02-26T05:41:30Z,2.1.0,,Vulnerability,Floating point exception,torch.Tensor.fmod,Floating point exception,N\A,Floating point exception (core dumped),https://github.com/pytorch/pytorch/issues/120597,API Input Context,Large Input Tensor,,
,yes,2024-02-07T23:09:01Z,2.1.0,,Logical,RuntimeError,torch.Tensor.stft,cuFFT error,N\A,RuntimeError: cuFFT error: CUFFT_INTERNAL_ERROR,https://github.com/pytorch/pytorch/issues/119420,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2024-02-07T07:18:45Z,2.1.0,,Logical,Inconsistent results,torch.onnx.dynamo_export,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/119360,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2024-02-07T07:18:45Z,2.1.0,,Logical,Inconsistent results,torch.fft.irfft,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/119360,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2024-01-25T17:13:39Z,2.0.1,,Logical,Inconsistent results,torch.Tensor.log10,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118303,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T17:13:39Z,2.0.1,,Logical,Inconsistent results,torch.nn.functional.mish,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118303,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T17:04:07Z,2.0.1,,Logical,Inconsistent results,torch.threshold_,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118302,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T17:04:07Z,2.0.1,,Logical,Inconsistent results,torch.adaptive_avg_pool1d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118302,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T16:56:56Z,2.0.1,,Logical,Inconsistent results,torch._C._linalg.linalg_ldl_factor_ex,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118300,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T16:56:56Z,2.0.1,,Logical,Inconsistent results,torch._C._special.special_spherical_bessel_j0,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118300,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T16:47:20Z,2.0.1,,Logical,Inconsistent results,torch.cosine_similarity,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118297,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-25T16:47:20Z,2.0.1,,Logical,Inconsistent results,torch._C._linalg.linalg_svdvals,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118297,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2024-01-24T02:51:36Z,2.1.0,,Logical,Inconsistent results,torch.nn.MultiheadAttention,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/118165,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2023-12-08T12:22:02Z,2.1.0,,Vulnerability,Segmentation fault,torch.nn.functional.embedding,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/115416,API Input Context,Large Input Tensor,,
,yes,2023-12-01T10:24:55Z,2.1.0,,Performance bug,Performance bug,torch.einsum,Tried to allocate,N\A,OutOfMemoryError,https://github.com/pytorch/pytorch/issues/114943,API Input Context,Modeling Parameter Space,,
DeepRel,yes,2023-11-28T07:52:42Z,2.1.0,,Logical,RuntimeError,torch.fft.ifft,Invalid number of data points,N\A,RuntimeError: Invalid number of data points (0) specified,https://github.com/pytorch/pytorch/issues/114672,API Input Context,Empty Input Tensor,,
,no,2023-11-28T00:35:47Z,2.1.0,,Logical,RuntimeError,torch.Tensor.conj,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !this_view_meta->has_fw_view() INTERNAL ASSERT FAILED at ""../torch/csrc/autograd/autograd_meta.cpp"":256, please report a bug to PyTorch. Expected the output of forward differentiable view operations to have the tangent have the same layout as primal",https://github.com/pytorch/pytorch/issues/114650,API Input Context,Modeling Parameter Space,,
,yes,2023-11-20T02:26:16Z,2.0.1,,Logical,RuntimeError,torch.nn.Conv2d,NestedTensorImpl doesn't support sizes,N\A,RuntimeError: Internal error: NestedTensorImpl doesn't support sizes. Please file an issue on https://github.com/pytorch/nestedtensor,https://github.com/pytorch/pytorch/issues/114090,API Input Context,Modeling Parameter Space,,
,no,2023-11-14T00:43:20Z,2.0.1,,Vulnerability,Segmentation fault,torch.utils.data.DataLoader,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/113600,API Input Context,Modeling Parameter Space,,
,no,2023-11-10T13:53:18Z,2.1.0,,Logical,AssertionError,torch.onnx.dynamo_export,raise,N\A,"    raise AssertionError(msg)
AssertionError: ",https://github.com/pytorch/pytorch/issues/113444,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-11-08T06:28:03Z,2.1.0,,Logical,RuntimeError,torch.cuda.nccl.all_gather,please report this issue,N\A,RuntimeError: NCCL Error 3: internal error - please report this issue to the NCCL developers,https://github.com/pytorch/pytorch/issues/113245,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2023-10-26T14:13:20Z,2.1.0,,Logical,Inconsistent results,torch.nn.parallel.DistributedDataParallel,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/112152,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-10-25T10:23:04Z,2.0.1,,Logical,Inconsistent results,torch.torch.nn.functional.scaled_dot_product_attention,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/112015,API Input Context,Modeling Parameter Space,,
,yes,2023-10-19T15:24:58Z,2.1.0,,Performance bug,Performance bug,torch.from_numpy,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/111570,Test Case Context,Oracle for Performance Bug Detection,,
,yes,2023-10-01T13:37:59Z,2.0.1,,Logical,Inconsistent results,torch.Tensor.dropout,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110356,API Input Context,Modeling Parameter Space,,
,yes,2023-10-01T13:37:59Z,2.0.1,,Logical,Inconsistent results,torch.nn.Dropout,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110356,API Input Context,Modeling Parameter Space,,
,yes,2023-10-01T13:37:59Z,2.0.1,,Logical,Inconsistent results,torch.nn.functional.dropout,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/110356,API Input Context,Modeling Parameter Space,,
,yes,2023-09-03T17:42:17Z,2.0.1,,Performance bug,Performance bug,torch.Tensor.cummax,Slow Execution,N\A,Slow Execution,https://github.com/pytorch/pytorch/issues/108496,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,no,2023-09-01T17:53:14Z,2.1.0,,Logical,Inconsistent results,torch.jit.trace,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/108432,Test Case Context,Modeling Multiple API Usage Pattern+Modeling Compile/Eager Execution Mode,,
,no,2023-08-29T14:43:21Z,2.0.1,,Logical,Others,torch.Tensor.uniform_,Breaks,N\A,Breaks,https://github.com/pytorch/pytorch/issues/108152,API Input Context,Modeling Parameter Space,,
,yes,2023-08-23T21:32:53Z,2.0.1,,Logical,AttributeError,torch.utils.tensorboard.writer.SummaryWriter.add_image_with_boxes,object has no attribute,N\A,AttributeError: 'ImageFont' object has no attribute 'getsize',https://github.com/pytorch/pytorch/issues/107820,API Input Context,Modeling Parameter Space,,
,yes,2023-08-19T15:01:32Z,2.0.1,,Logical,Inconsistent results,torch.nn.Conv2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107514,API Input Context,Modeling Specific Data Types,,
,no,2023-08-16T15:12:16Z,2.1.0,,Logical,Inconsistent results,torch.distributed.destroy_process_group,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107298,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-08-14T09:18:31Z,2.0.1,,Logical,Inconsistent results,torch.vmap,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/107125,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,no,2023-08-11T06:25:40Z,2.1.0,,Logical,RuntimeError,torch.onnx.dynamo_export,expected at most,N\A,"RuntimeError: forward() expected at most 2 argument(s) but received 5 argument(s). Declaration: forward(__torch__.DataCov self, Tensor x1) -> Tensor",https://github.com/pytorch/pytorch/issues/107016,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-08-07T04:34:47Z,2.0.1,,Logical,Inconsistent results,torch.nn.parallel.DistributedDataParallel,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106690,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-07-29T06:31:31Z,2.1.0,,Logical,Inconsistent results,torch.nn.MultiMarginLoss,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/106251,API Input Context,Modeling Parameter Space,,
,no,2023-07-29T01:03:48Z,2.0.1,,Logical,Others,torch.optim.lr_scheduler.OneCycleLR,Others,N\A,Others,https://github.com/pytorch/pytorch/issues/106243,Test Case Context,Others,,
,yes,2023-07-10T12:19:05Z,2.0.1,,Logical,Inconsistent results,torch.fft.rfftn,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/104867,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-07-06T20:08:35Z,2.1.0,,Logical,RuntimeError,torch.ops.aten.sub,Arguments for call are not valid,N\A,RuntimeError:  Arguments for call are not valid. The following variants are available:,https://github.com/pytorch/pytorch/issues/104732,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023-06-07T17:18:19Z,2.0.0,,Logical,RuntimeError,torch.nn.LSTM,Cannot access data pointer,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/103161,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-06-07T17:18:19Z,2.0.0,,Logical,RuntimeError,torch.func.jacfwd,Cannot access data pointer,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/103161,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-06-07T17:18:19Z,2.0.0,,Logical,RuntimeError,torch.func.jacrev,Cannot access data pointer,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/103161,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-06-07T17:18:19Z,2.0.0,,Logical,RuntimeError,torch.func.functional_call,Cannot access data pointer,N\A,RuntimeError: Cannot access data pointer of Tensor that doesn't have storage,https://github.com/pytorch/pytorch/issues/103161,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-06-05T03:16:15Z,2.0.1,,Logical,Unsupported Error,torch.Tensor.cumprod,Exporting the operator,N\A,torch.onnx.errors.UnsupportedOperatorError: Exporting the operator 'aten::cumprod' to ONNX opset version 14 is not supported.,https://github.com/pytorch/pytorch/issues/102948,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-05-23T13:50:21Z,2.0.0,,Logical,Others,torch.nn.BatchNorm2d,symbolically traced variables,N\A,TraceError: symbolically traced variables cannot be used as inputs to control flow,https://github.com/pytorch/pytorch/issues/102085,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-05-11T06:15:29Z,2.0.0,,Logical,Inconsistent results,torch.cuda.memory_summary,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/101159,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-05-07T22:18:48Z,2.1.0,,Logical,Inconsistent results,torch.ops.aten.baddbmm,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/100838,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Compile/Eager Execution Mode,,
,yes,2023-04-14T13:59:29Z,2.0.0,,Logical,Inconsistent results,torch.nn.Conv2d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/99149,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-04-14T13:59:29Z,2.0.0,,Logical,Inconsistent results,torch.nn.Conv1d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/99149,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-04-14T13:59:29Z,2.0.0,,Logical,Inconsistent results,torch.nn.Conv3d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/99149,Test Case Context,Modeling Multiple API Usage Pattern,,
,no,2023-04-12T15:19:18Z,2.0.0,,Logical,Inconsistent results,torch.Tensor.dist,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/issues/98939,API Input Context,Negative Integer Argument,,
,no,2023-03-29T21:40:31Z,2.0.0,,Logical,ValueError,torch.stack,Cannot determine scalar type,N\A,"torch.onnx.errors.SymbolicValueError: Cannot determine scalar type for this '<class 'torch.TensorType'>' instance and a default value was not provided.  [Caused by the value '21 defined in (%21 : Tensor = onnx::Sub(%input, %20), scope: __main__.Example::/torch.nn.modules.normalization.LayerNorm::layer_norm",https://github.com/pytorch/pytorch/issues/97913,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-03-26T08:17:50Z,2.0.0,,Logical,TypeError,torch.nn.GroupNorm,unsupported operand,N\A,TypeError: unsupported operand type(s) for divmod(): 'SymInt' and 'int',https://github.com/pytorch/pytorch/issues/97623,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,yes,2023-03-21T21:39:08Z,2.1.0,,Logical,RuntimeError,torch.func.jacfwd,Sparse CSR tensors,N\A,RuntimeError: Sparse CSR tensors do not have strides,https://github.com/pytorch/pytorch/issues/97286,API Input Context,Modeling Parameter Space,,
,yes,2023-03-21T20:52:00Z,2.1.0,,Logical,RuntimeError,torch.func.jacfwd,required rank 4 tensor,N\A,RuntimeError: required rank 4 tensor to use channels_last format,https://github.com/pytorch/pytorch/issues/97283,API Input Context,Modeling Parameter Space,,
ace_1,yes,2023-03-18T10:58:24Z,2.1.0,,Vulnerability,Segmentation fault,torch.Tensor.adjoint,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/97087,API Input Context,Large Input Tensor,,
ace_1,yes,2023-03-18T10:53:40Z,2.1.0,,Vulnerability,Floating point exception,torch.Tensor.index_select,Floating point exception,N\A,Floating point exception (core dumped),https://github.com/pytorch/pytorch/issues/97086,API Input Context,Empty Input Tensor,,
,yes,2023-03-15T00:30:22Z,2.0.0,,Logical,RuntimeError,torch.Tensor.cumprod,INTERNAL ASSERT FAILED,N\A,RuntimeError: t == DeviceType::CUDA INTERNAL ASSERT FAILED,https://github.com/pytorch/pytorch/issues/96789,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023-03-08T18:29:32Z,2.0.0,,Logical,RuntimeError,torch.nn.FractionalMaxPool3d,INTERNAL ASSERT FAILED,N\A,RuntimeError: index >= 0 && index < inputT * inputH * inputW INTERNAL ASSERT FAILED at ,https://github.com/pytorch/pytorch/issues/96316,Test Case Context,Modeling Multiple API Usage Pattern,,
ace_1,yes,2023-03-08T06:10:16Z,2.1.0,,Vulnerability,Segmentation fault,torch.adaptive_max_pool1d,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/96277,API Input Context,Large Input Tensor,,
ace_1,yes,2023-03-08T06:06:12Z,2.1.0,,Vulnerability,Segmentation fault,torch.Tensor.geqrf,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/96276,API Input Context,Large Input Tensor,,
ace_1,yes,2023-03-08T05:59:47Z,2.1.0,,Vulnerability,Segmentation fault,torch.Tensor.pinverse,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/96275,API Input Context,Large Input Tensor,,
,no,2023-02-22T19:34:45Z,2.0.0,,Logical,RuntimeError,torch.distributed.Store,INTERNAL ASSERT FAILED,N\A,RuntimeError: fn INTERNAL ASSERT FAILED at ,https://github.com/pytorch/pytorch/issues/95309,API Input Context,Modeling Parameter Space,,
ace_1,yes,2023-02-14T09:09:10Z,2.0.0,,Vulnerability,Segmentation fault,torch.lstm,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94808,API Input Context,Modeling Parameter Space,,
,yes,2023-02-12T16:32:30Z,2.0.0,,Logical,RuntimeError,torch.where,INTERNAL ASSERT FAILED,N\A,RuntimeError: !needs_dynamic_casting<func_t>::check(iter) INTERNAL ASSERT FAILED ,https://github.com/pytorch/pytorch/issues/94704,API Input Context,Modeling Specific Data Types,,
ace_1,yes,2023-02-12T06:25:54Z,2.0.0,,Vulnerability,Segmentation fault,torch.avg_pool1d,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94698,API Input Context,Large Input Tensor,,
ace_1,yes,2023-02-12T06:10:39Z,2.0.0,,Vulnerability,Segmentation fault,torch.max_pool1d_with_indices,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94696,API Input Context,Large Input Tensor,,
ace_1,yes,2023-02-11T08:51:26Z,2.0.0,,Vulnerability,Segmentation fault,torch.Tensor.svd,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94669,API Input Context,Large Input Tensor,,
ace_1,yes,2023-02-11T08:48:00Z,2.0.0,,Vulnerability,Segmentation fault,torch.lstm,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94668,API Input Context,Modeling Parameter Space,,
ace_1,yes,2023-02-10T13:50:03Z,2.0.0,,Vulnerability,Segmentation fault,torch.histogramdd,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/issues/94594,API Input Context,Modeling Parameter Space,,
,yes,2023-02-08T22:12:06Z,2.0.0,,Logical,RuntimeError,torch.cuda.jiterator._create_multi_output_jit_fn,INTERNAL ASSERT FAILED,N\A,RuntimeError: t == DeviceType::CUDA INTERNAL ASSERT FAILED at ,https://github.com/pytorch/pytorch/issues/94443,API Input Context,Others,,
,yes,2023-02-06T19:30:11Z,2.0.0,,Logical,RuntimeError,torch.zeros_like,output with shape,N\A,"RuntimeError: output with shape [1, 3, 3] doesn't match the broadcast shape [9, 3, 3]",https://github.com/pytorch/pytorch/issues/94208,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-02-04T01:58:14Z,2.0.0,,Logical,RuntimeError,torch.cat,Tensors must have same number of dimensions,N\A,RuntimeError: Tensors must have same number of dimensions: got 5 and 2,https://github.com/pytorch/pytorch/issues/94115,API Input Context,Empty Input Tensor,,
,yes,2023-02-04T01:32:54Z,2.0.0,,Logical,RuntimeError,torch.svd,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !this_view_meta->has_fw_view() INTERNAL ASSERT FAILED at ""/opt/conda/conda-bld/pytorch_1672906354936/work/torch/csrc/autograd/autograd_meta.cpp"":256, please report a bug to PyTorch.  Expected the output of forward differentiable view operations to have the tangent have the same layout as primal",https://github.com/pytorch/pytorch/issues/94111,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023-02-03T22:57:22Z,2.0.0,,Logical,IndexError,torch.Tensor.unfold,Dimension out of range,N\A,"IndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)",https://github.com/pytorch/pytorch/issues/94085,API Input Context,Empty Input Tensor,,
,no,2023-01-25T19:16:35Z,2.0.0,,Logical,TypeError,torch.fx.replace_pattern,incompatible function arguments,N\A,"TypeError: ln_fwd(): incompatible function arguments. The following argument types are supported: 1. (arg0: torch.Tensor, arg1: torch.Tensor, arg2: torch.Tensor, arg3: float) -> List[torch.Tensor]",https://github.com/pytorch/pytorch/issues/93002,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023-01-18T23:25:40Z,2.0.0,,Logical,RuntimeError,torch.Tensor.addcmul_,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !needs_dynamic_casting<func_t>::check(iter) INTERNAL ASSERT FAILED at ""/opt/conda/conda-bld/pytorch_1672906354936/work/aten/src/ATen/native/cpu/Loops.h"":347, please report a bug to PyTorch. ",https://github.com/pytorch/pytorch/issues/92594,API Input Context,Modeling Specific Data Types,,
,yes,2023-01-18T23:25:40Z,2.0.0,,Logical,Others,torch.Tensor.addcmul_,Jacobian mismatch for output,N\A,"GradcheckError: Jacobian mismatch for output 0 with respect to input 0,",https://github.com/pytorch/pytorch/issues/92594,API Input Context,Modeling Specific Data Types,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.Tensor.mul,Expected a value of type,N\A,Torch.compile Error: RuntimeError: aten::_conj() Expected a value of type 'Tensor' for argument 'self' but instead found type 'complex'.,https://github.com/pytorch/pytorch/commit/5570a0da0af369768083875a354b70cd72f0ff07,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.linalg.lstsq,INTERNAL ASSERT FAILED,N\A,"RuntimeError: input.size(-2) == other_2d.size(-2) INTERNAL ASSERT FAILED at ""/home/yonghyeon/pytorch/pytorch-asan/aten/src/ATen/native/BatchLinearAlgebra.cpp"":3364, please report a bug to PyTorch.",https://github.com/pytorch/pytorch/commit/f422027fceb71b1f2f7d940f3ade32ced2ed0ba0,API Input Context,Inputs with Incompatible Shapes,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.vmap,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !dynamicLayerStack.empty() INTERNAL ASSERT FAILED at ""/ho",https://github.com/pytorch/pytorch/commit/b38de2f9e21860dc12cccd0ee0d320df54aeaf18,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.compile,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !dynamicLayerStack.empty() INTERNAL ASSERT FAILED at ""/ho",https://github.com/pytorch/pytorch/commit/b38de2f9e21860dc12cccd0ee0d320df54aeaf18,Test Case Context,Modeling Multiple API Usage Pattern,,
,no,2024,2.1.0,,Logical,ValueError,torch.jit.load,vector::_M_default_append,N\A,ValueError: vector::_M_default_append,https://github.com/pytorch/pytorch/commit/f343f98710dfa7305a873f558086c595a3c3d3d4,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Logical,TypeError,torch.optim.SparseAdam,params argument given to the optimizer,N\A,TypeError: params argument given to the optimizer should be an iterable of Tensors or dicts,https://github.com/pytorch/pytorch/commit/665637714f46f6dcc7f6d2ad860184e6335fe35a,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.tensor,INTERNAL ASSERT FAILED,N\A,"RuntimeError: false INTERNAL ASSERT FAILED at ""/ansible/pytorch/torch/csrc/lazy/core/hash.h"":139, please report a bug to PyTorch. Unsupported scalar type:UInt16",https://github.com/pytorch/pytorch/commit/7dfd2949d74ab3a523a0abc6270cc426cdcf4a00,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2024,2.1.0,,Logical,Inconsistent results,torch.multiprocessing.ProcessContext.join,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/92eb1731d4cdc9f15cf9304477a5701415453227,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2024,2.0.0,,Vulnerability,Others,torch.nn.functional.binary_cross_entropy,Fatal Python error: Aborted,Aborted,LLVM ERROR: Failed to infer result type(s). Fatal Python error: Aborted,https://github.com/pytorch/pytorch/commit/5677128cb892d17fe2281beae9e394fd6f89e455,API Input Context,Modeling Specific Data Types,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.Tensor.where,INTERNAL ASSERT FAILED,N\A,"RuntimeError: !needs_dynamic_casting<func_t>::check(iter) INTERNAL ASSERT FAILED at ""../aten/src/ATen/native/cpu/Loops.h"":310, please report a bug to PyTorch.",https://github.com/pytorch/pytorch/commit/1866ee6735adb53b8c8fcc46186d74209e8d5496,API Input Context,Empty Input Tensor,,
,yes,2024,2.1.0,,Vulnerability,Segmentation fault,torch._C._nn.replication_pad2d,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/commit/69cedc16c593e777cd36f064dd862c78d6291c0b,API Input Context,Empty List Argument,,
,yes,2024,2.1.0,,Vulnerability,Segmentation fault,torch._C._nn.replication_pad3d,Segmentation fault,N\A,Segmentation fault (core dumped),https://github.com/pytorch/pytorch/commit/69cedc16c593e777cd36f064dd862c78d6291c0b,API Input Context,Empty List Argument,,
,yes,2024,2.0.0,,Vulnerability,Floating point exception,torch.native_channel_shuffle,Floating point exception,N\A,Floating point exception (core dumped),https://github.com/pytorch/pytorch/commit/34e3f6f3c9ed0b8ebdbf37637722a6e53274198a,API Input Context,Empty Input Tensor,,
,yes,2024,2.1.0,,Logical,RuntimeError,torch.jit.trace,Named tensors are not supported with the tracer,N\A,RuntimeError: NYI: Named tensors are not supported with the tracer,https://github.com/pytorch/pytorch/commit/939008a268cfc345d758a51c13a2fd9fe8327a84,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Logical,Inconsistent results,torch.signal.windows.kaiser,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/534c73d478fc967cb3a6a29f5eada94bc4ce2c29,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Logical,AttributeError,torch.nn.TransformerEncoderLaye,NoneType' object has no attribute 'device',N\A,AttributeError: 'NoneType' object has no attribute 'device',https://github.com/pytorch/pytorch/commit/d0cf2182eac1c87805ceef4416397e9049100685,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Vulnerability,Segmentation fault,torch.Tensor.all,stack-overflow on address,N\A,1061683==ERROR: AddressSanitizer: stack-overflow on address 0x7ffff14a5d30 (pc 0x7fe8968c37a4 bp 0x7ffff14a6570 sp 0x7ffff14a5d30 T0,https://github.com/pytorch/pytorch/commit/6793b9910787c388c156e0dcf630e781af7a17f5,API Input Context,Modeling Parameter Space,,
,yes,2024,2.1.0,,Vulnerability,Segmentation fault,torch.Tensor.any,stack-overflow on address,N\A,1061683==ERROR: AddressSanitizer: stack-overflow on address 0x7ffff14a5d30 (pc 0x7fe8968c37a4 bp 0x7ffff14a6570 sp 0x7ffff14a5d30 T0,https://github.com/pytorch/pytorch/commit/6793b9910787c388c156e0dcf630e781af7a17f5,API Input Context,Modeling Parameter Space,,
,no,2023,2.0.0,,Logical,BackendCompilerFailed,torch.ops.aten.div,raised,N\A,torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised,https://github.com/pytorch/pytorch/commit/b72127cd4b56a4e6978ec3d46bc8daf7c75e0c08,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023,2.1.0,,Logical,RuntimeError,torch.nn.Linear,failed to parse the list of possible processors,N\A,Error in cpuinfo: failed to parse the list of possible processors in /sys/devices/system/cpu/possible,https://github.com/pytorch/pytorch/commit/4123cca859037b6ac86f83cdb0b6030eac345c6f,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Others,torch.Tensor.erfinv,Others,N\A,Others,https://github.com/pytorch/pytorch/commit/74dfdc567bd90301e1db0550683c184075bdb4e8,API Input Context,Modeling Parameter Space,,
TitanFuzz,yes,2023,2.0.0,,Logical,RuntimeError,torch.nn.CTCLoss,Expected tensor to have,N\A,"RuntimeError: Expected tensor to have CPU Backend, but got tensor with CUDA Backend (while checking arguments for cudnn_ctc_loss)",https://github.com/pytorch/pytorch/commit/1becd2c314f45bded8d3fbec91d785e7190b4afe,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023,2.0.0,,Logical,Others,torch.autograd.gradcheck,Jacobian mismatch for output,N\A,"torch.autograd.gradcheck.GradcheckError: Jacobian mismatch for output 0 with respect to input 0,",https://github.com/pytorch/pytorch/commit/8739d1e3f9b08f4282fe79fc8dacd781d16913ff,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,BackendCompilerFailed,torch.nn.functional.conv2d,raised,N\A,torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised: LoweringException: AssertionError:,https://github.com/pytorch/pytorch/commit/85aa3723749e0d06aa5fd34215b9b93529a60995,API Input Context,Modeling Parameter Space,,
,no,2023,2.0.0,,Logical,Others,torch.optim.lr_scheduler.CyclicLR,Can't pickle,N\A,_pickle.PicklingError: Can't pickle <function <lambda> at 0x000001A51DF67600>: attribute lookup <lambda> on __main__ failed,https://github.com/pytorch/pytorch/commit/2b72543f3667096f3e41ddab528af14191c15ec3,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,RuntimeError,torch.Tensor.add,Failed running,N\A,"TorchRuntimeError: Failed running call_function <built-in method add of type object at 0x7cda83f9a360>(*(FakeTensor(..., size=(4, 5), dtype=torch.int32), 7), **{'alpha': 3}): Tensor with dtype torch.int64 is not the expected dtype of torch.int32!",https://github.com/pytorch/pytorch/commit/c77a4a409654dbc0ac4a528c37873b0acb1be32d,Test Case Context,Model Tensor Execution on CUDA Devices,,
,no,2023,2.0.0,,Logical,RuntimeError,torch.set_num_threads,Overflow when unpacking long,N\A,RuntimeError: Overflow when unpacking long,https://github.com/pytorch/pytorch/commit/78f3937ee84e71475942598f4b51dce7c8a70783,API Input Context,Large Integer Argument,,
,yes,2023,2.0.0,,Logical,Inconsistent results,torch.cuda.memory_stats,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/0b424ee0b7bfe09e0a438a63e8336e95eea85901,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Vulnerability,Buffer Overflow,torch.Tensor.sort,Out of Bound,N\A,Out of Bound Writes,https://github.com/pytorch/pytorch/commit/03da0694b7f414f8124d6a1e377e1a7484e0cfb6,API Input Context,Large Input Tensor,,
,yes,2023,2.1.0,,Logical,Others,torch.nn.LeakyReLU,name 'buf0' is not defined,N\A,NameError: name 'buf0' is not defined,https://github.com/pytorch/pytorch/commit/1a361e4e9ff1a8dadd72c7301441cfb83d687f24,API Input Context,Empty Input Tensor,,
,no,2023,2.0.0,,Logical,Others,torch.compile,Final fx graph captures only empty,N\A,"Final fx graph captures only empty
graph():
    %arg0_1 : [#users=0] = placeholder[target=arg0_1]
    %empty : [#users=1] = call_function[target=torch.ops.aten.empty.memory_format](args = ([2, 3],), kwargs = {dtype: torch.float32, layout: torch.strided, device: hpu:0, pin_memory: False})
    return (empty,)",https://github.com/pytorch/pytorch/commit/1b3dc05c3e703841e64e0277d473a0baf3296671,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023,2.1.0,,Logical,TypeError,torch.quantized_max_pool1d,failed to unpack the object at pos,N\A,"TypeError: max_pool1d(): argument 'dilation' failed to unpack the object at pos 1 with error ""Overflow when unpacking long""",https://github.com/pytorch/pytorch/commit/6f0d0b3850db984e73aad91daf75e65272301e8d,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,TypeError,torch.quantized_max_pool2d,failed to unpack the object at pos,N\A,"TypeError: max_pool1d(): argument 'dilation' failed to unpack the object at pos 1 with error ""Overflow when unpacking long""",https://github.com/pytorch/pytorch/commit/6f0d0b3850db984e73aad91daf75e65272301e8d,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,TypeError,torch.nn.functional.max_pool1,failed to unpack the object at pos,N\A,"TypeError: max_pool1d(): argument 'dilation' failed to unpack the object at pos 1 with error ""Overflow when unpacking long""",https://github.com/pytorch/pytorch/commit/6f0d0b3850db984e73aad91daf75e65272301e8d,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,TypeError,torch.nn.functional.max_pool2,failed to unpack the object at pos,N\A,"TypeError: max_pool1d(): argument 'dilation' failed to unpack the object at pos 1 with error ""Overflow when unpacking long""",https://github.com/pytorch/pytorch/commit/6f0d0b3850db984e73aad91daf75e65272301e8d,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,TypeError,torch.nn.functional.max_pool3,failed to unpack the object at pos,N\A,"TypeError: max_pool1d(): argument 'dilation' failed to unpack the object at pos 1 with error ""Overflow when unpacking long""",https://github.com/pytorch/pytorch/commit/6f0d0b3850db984e73aad91daf75e65272301e8d,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,RuntimeError,torch.nn.AdaptiveMaxPool1d,Expected input to have non-zero size,N\A,"RuntimeError: adaptive_max_pool2d(): Expected input to have non-zero size for non-batch dimensions, but input has sizes [0, 1, 1, 0] with dimension 3 being empty",https://github.com/pytorch/pytorch/commit/3b160ecc71eaed3db64cac53ef9089bea876b6d3,API Input Context,Empty Input Tensor,,
,no,2023,2.0.1,,Performance bug,Performance bug,torch.onnx.export,Memory Leak,N\A,Memory Leak,https://github.com/pytorch/pytorch/commit/2b3917dc63f7daf197602f3f924a1bb1c0c94802,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2023,2.1.0,,Logical,RuntimeError,torch.Tensor.index_put_,operation not permitted when stream is,N\A,RuntimeError: CUDA error: operation not permitted when stream is capturing. Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.,https://github.com/pytorch/pytorch/commit/5abc5ab55dc5fd7bc0d199c5de8518f8e503515c,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023,2.0.0,,Logical,Others,torch.Tensor.resize_,Others,N\A,Others,https://github.com/pytorch/pytorch/commit/0ccdbbe2337b0955a41407a7b31991772387abb0,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,RuntimeError,torch.Tensor.squeeze,CUDA kernel errors might be,N\A,"RuntimeError: CUDA error: device-side assert triggered CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.",https://github.com/pytorch/pytorch/commit/9626604bdd313901524d8e8d0c6c6bef427d82f1,API Input Context,Modeling Parameter Space,,
,no,2023,2.0.0,,Logical,Inconsistent results,torch.distributed.all_reduce,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/9165d46b8949f38744c9c642172767cf5833b292,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Vulnerability,Others,torch.to_dtype,IOT instruction (core dumped),N\A,IOT instruction (core dumped),https://github.com/pytorch/pytorch/commit/adcefcb3781b29ed0a1430b66ccf5d5d8ae47dda,API Input Context,Modeling Specific Data Types,,
,yes,2023,2.1.0,,Logical,RuntimeError,torch.sparse_csr_tensor,is not satisfied,N\A,RuntimeError: `0 <= col_indices < ncols` is not satisfied.,https://github.com/pytorch/pytorch/commit/9f97b7c43b035b4ba6e40541d7ed50d116d66aa2,API Input Context,Large Input Tensor,,
,yes,2023,2.1.0,,Logical,Inconsistent results,torch.Tensor.neg,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/e5b7c7a04fae8c58de30a647d7e879f4ffc9eda2,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Inconsistent results,torch.nn.MaxPool1d,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/563d8058f44d5a6324893b4187b91c4cfb134aa7,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2023,2.0.0,,Logical,RuntimeError,torch.nn.LSTM,expected scalar type Long but found Float,N\A,RuntimeError: expected scalar type Long but found Float,https://github.com/pytorch/pytorch/commit/022119879065990ba6b65fbed17e97dc0e79f2f4,API Input Context,Modeling Parameter Space,,
,yes,2023,2.1.0,,Logical,AssertionError,torch.cat,assert,N\A,assert torch.allclose(AssertionError: ,https://github.com/pytorch/pytorch/commit/bb4e9e9124a81859de6ac1ef0c0798643cb9b7a6,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023,2.1.0,,Logical,ValueError,torch.Tensor.split,add up to the tensor's size in the given dimension,N\A,ValueError: Split sizes don't add up to the tensor's size in the given dimension,https://github.com/pytorch/pytorch/commit/41069f2faa726faa8f9ca633391453d16f346e11,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Inconsistent results,torch.segment_reduce,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/efed5a4969c38da406b033af05db81d5d3ded90a,API Input Context,Zero Input Tensor,,
,yes,2023,2.0.0,,Logical,RuntimeError,torch.nn.FractionalMaxPool2d,kernel_size must either be a single Int or tuple of Ints,N\A,RuntimeError: fractional_max_pool2d: kernel_size must either be a single Int or tuple of Ints,https://github.com/pytorch/pytorch/commit/dbf0db958fa44a92d7685dd0128d76fc58089516,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Inconsistent results,torch.tensor,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/a8429342df25531f39f2b16a934a082566d44da1,API Input Context,Modeling Specific Data Types,,
,yes,2023,2.0.0,,Logical,IndexError,torch.is_floating_point,list index out of range,N\A,IndexError: list index out of range,https://github.com/pytorch/pytorch/commit/9c7b03d51e097c40abdb9ef55ce79b98d7f1fd48,API Input Context,Large Input Tensor,,
,yes,2023,2.0.0,,Logical,RuntimeError,torch.masked_scatter,expected BoolTensor or ByteTensor for mask,N\A,RuntimeError: masked_select: expected BoolTensor or ByteTensor for mask,https://github.com/pytorch/pytorch/commit/2af09393f936cb646fea03bdedc8d7429f1120f6,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Others,torch.nn.TransformerEncoderLayer,Others,N\A,Others,https://github.com/pytorch/pytorch/commit/c757647dd8c198fbcca4d4e6c7cba6f1d12b97e6,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,Others,torch.nn.functional.embedding_bag,error,N\A,Others,https://github.com/pytorch/pytorch/commit/29b216acd5f8db0968e0de85b4515fce215598ed,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,no,2023,2.1.0,,Logical,Inconsistent results,torch.ops.aten.unsqueeze_,Inconsistent results,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/6ae06e49ac92442e583f05e6b88f58670cecebaa,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,BackendCompilerFailed,torch.ops.aten.mean.dim,inductor raised CppCompileError,N\A,torch._dynamo.exc.BackendCompilerFailed: inductor raised CppCompileError: C++ compile error,https://github.com/pytorch/pytorch/commit/1d7133c542fd5f87cd4efd87da449d4484050aaa,API Input Context,Modeling Parameter Space,,
,yes,2023,2.0.0,,Logical,RuntimeError,torch.Tensor.var,returned an invalid gradient at index,N\A,"RuntimeError: Function VarBackward0 returned an invalid gradient at index 0 - got [] but expected shape compatible with [1, 1]",https://github.com/pytorch/pytorch/commit/92f3feabaa919767229c51adc76e2f7ff2d74ad0,Test Case Context,Modeling Multiple API Usage Pattern,,
,yes,2023,2.0.0,,Vulnerability,Buffer Overflow,torch.Tensor.index_select,heap-buffer-overflow on address,N\A,100125==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x609000000284 at pc 0x7fb6f1b2e155 bp 0x7fff89752850 sp 0x7fff89752848 WRITE of size 4 at 0x609000000284 thread T0,https://github.com/pytorch/pytorch/commit/db6cfff827158c3f22a6707aed35d2013ec2d994,API Input Context,Empty Input Tensor,,
,yes,2022,2.1.0,,Logical,RuntimeError,torch.nn.functional.conv2d,on tensor with symbolic,N\A,RuntimeError: Cannot call sizes() on tensor with symbolic sizes/strides,https://github.com/pytorch/pytorch/commit/795473ff5e861f21c6b5e0611177fcfa9e1c4e0c,API Input Context,Modeling Parameter Space,,
,yes,2022,2.0.0,,Logical,RuntimeError,torch.nested.nested_tensor,Internal error:,N\A,RuntimeError: Internal error: NestedTensorImpl doesn't support sizes. Please file an issue on https://github.com/pytorch/nestedtensor,https://github.com/pytorch/pytorch/commit/1cad744694d7feb7c55e5f4ff4a6ae749686bfb5,API Input Context,Modeling Parameter Space,,
,no,2022,2.1.0,,Logical,AssertionError,torch.jit.trace,Graphs differed across invocations,N\A,ERROR: Graphs differed across invocations! Graph diff:,https://github.com/pytorch/pytorch/commit/ece0002c4beaebaf083dc75b7bf8ceb19edf7a0b,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Multiple API Usage Pattern,,
,no,2022,2.1.0,,Logical,TypeError,torch.nn.LazyLinear,,N\A,"shenxiu@devbig1109:fbcode  (5c57dd833)$ bento console --kernel pytorch --local
/data/users/shenxiu/fbsource/buck-out/v2/gen/fbcode/26f2c80c27f9e71d/bento/kernels/__bento_kernel_pytorch__/bento_kernel_pytorch#link-tree/scribeutil/lib.py:9: DeprecationWarning: The ""thrift"" clients in libfb.py.thrift_clients are not proper thrift clients, and often have unexpected or incorrect behaviour. They are also completely unsupported. Please use a supported client from https://fburl.com/srpy or a supported raw thrift client if you cannot use ServiceRouter.",https://github.com/pytorch/pytorch/commit/ec714e33a3365ce405e2ae048c8adaa2751b9dba,Test Case Context,Modeling Complex Tensor Operations and Layer Interactions,,
,yes,2022,2.0.0,,Logical,Inconsistent results,torch.nn.Conv2d,or a supported raw thrift client if you cannot use ServiceRouter,N\A,Inconsistent results,https://github.com/pytorch/pytorch/commit/b019a416741c1a78ebed89516c1f9fdbfb276898,API Input Context,Modeling Parameter Space,,
,yes,2022,2.1.0,,Logical,Inconsistent results,torch.rand,object is not subscriptable,N\A,"/home/xxx/.local/lib/python3.7/site-packages/torch/onnx/utils.py:366: UserWarning: Skipping _decide_input_format
 'NoneType' object is not subscriptable
  warnings.warn(""Skipping _decide_input_format\n {}"".format(e.args[0]))
PyTorch Output DTypes: (torch.float64, torch.float32)
ONNX Output DTypes: ('onnx.TensorProto.FLOAT', 'onnx.TensorProto.FLOAT')",https://github.com/pytorch/pytorch/commit/1a845579b67f522e22d899126c92672f46f6a7b9,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2022,2.1.0,,Logical,RuntimeError,torch.nn.GroupNorm,Expected number of channels,N\A,"RuntimeError: Expected number of channels in input to be divisible by num_groups, but got input of shape [1, 6, 4, 4] and num_groups=5",https://github.com/pytorch/pytorch/commit/4a0f6e6c530b624bb5a4fbcaebe3fd43b1ff66c3,API Input Context,Modeling Parameter Space,,
,yes,2022,2.1.0,,Vulnerability,Others,torch.autograd.Function,Deadlock,N\A,Others,https://github.com/pytorch/pytorch/commit/b2a55076546021fd059794676e9066f361b3a091,Test Case Context+API Input Context,Modeling Complex Tensor Operations and Layer Interactions+Modeling Parameter Space,,
,yes,2022,2.0.0,,Vulnerability,Segmentation fault,torch.set_default_dtype,Segmentation fault,N\A,segmentation fault,https://github.com/pytorch/pytorch/commit/75db05c3fdb6f3b6682dab1532cd6e1e5b423527,API Input Context,Modeling Specific Data Types,,
,yes,2021,2.0.0,,Logical,RuntimeError,torch.save,Trying to resize storage that is not resizable,N\A,RuntimeError: Trying to resize storage that is not resizable at ../aten/src/TH/THStorageFunctions.cpp:100,https://github.com/pytorch/pytorch/commit/b69155f754bb077acf0dc3e32251f0352234ed5c,API Input Context,Modeling Specific Data Types,,
,yes,2021,2.1.0,,Logical,RuntimeError,torch.nn.GRU,stack expects a non-empty TensorList,N\A,RuntimeError: stack expects a non-empty TensorList,https://github.com/pytorch/pytorch/commit/79dc500a992b7d6f8eba96bc28ff856f2ea9e10d,API Input Context,Zero Integer Argument,,
,yes,2021,2.1.0,,Logical,RuntimeError,torch.Tensor,legacy constructor for device type,N\A,"RuntimeError: legacy constructor for device type: cpu was passed device type: cuda, but device type must be: cpu",https://github.com/pytorch/pytorch/commit/e8574b84bfa4dd132b6d451b05a9197856d7fda1,Test Case Context,Model Tensor Execution on CUDA Devices,,
,yes,2021,2.1.0,,Logical,RuntimeError,torch.Tensor.where,Expected condition to have ScalarType Byte,N\A,"RuntimeError: Expected condition to have ScalarType Byte, but got ScalarType Float",https://github.com/pytorch/pytorch/commit/f0f69c5dc104b2a433d582c5e8a71812b1a2ff6f,API Input Context,Modeling Specific Data Types,,
,yes,2021,2.0.0,,Logical,RuntimeError,torch.normal,but found std,N\A,"RuntimeError: normal_ expects std > 0.0, but found std=-1",https://github.com/pytorch/pytorch/commit/6c235ef267c350225b7ca3f0caedb1a05c9df9ec,API Input Context,Negative Integer Argument,,
,no,2021,2.0.0,,Logical,Others,torch.device,opencl:0,N\A,opencl:0,https://github.com/pytorch/pytorch/commit/751c30038feee49308abd71252beb69387d961d2,Test Case Context,Modeling Compile/Eager Execution Mode,,
DeepRel,yes,2020,2.0.0,,Logical,TypeError,torch.empty,must be tuple of ints,N\A,"TypeError: empty(): argument 'size' must be tuple of ints, but found element of type int at pos 1",https://github.com/pytorch/pytorch/commit/c675727adf36bdbb60933c9c7529d3ee34462093,API Input Context,Large Integer Argument,,
